<!doctype html><html lang=en dir=auto><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=robots content="index, follow"><title>Issue 15 | Folded Weekly</title><meta name=keywords content><meta name=description content="This week there&rsquo;s some more thoughts on AlphaFold and intrinsic disorder, a new experimental approach to look at high-energy protein conformations, and a structural angle to population genetics"><meta name=author content><link rel=canonical href=https://folded-weekly.netlify.app/posts/15/><link crossorigin=anonymous href=/assets/css/stylesheet.min.0caf3925a7f1eae074a9aadc5d80291998cf748272cb5b0e65ca8199d38d7909.css integrity="sha256-DK85Jafx6uB0qarcXYApGZjPdIJyy1sOZcqBmdONeQk=" rel="preload stylesheet" as=style><link rel=preload href=logo_full.png as=image><script defer crossorigin=anonymous src=/assets/js/highlight.min.b95bacdc39e37a332a9f883b1e78be4abc1fdca2bc1f2641f55e3cd3dabd4d61.js integrity="sha256-uVus3DnjejMqn4g7Hni+Srwf3KK8HyZB9V4809q9TWE=" onload=hljs.initHighlightingOnLoad()></script>
<link rel=icon href=https://folded-weekly.netlify.app/favicon.ico><link rel=icon type=image/png sizes=16x16 href=https://folded-weekly.netlify.app/favicon-16x16.png><link rel=icon type=image/png sizes=32x32 href=https://folded-weekly.netlify.app/favicon-32x32.png><link rel=apple-touch-icon href=https://folded-weekly.netlify.app/apple-touch-icon.png><link rel=mask-icon href=https://folded-weekly.netlify.app/favicon-16x16.png><meta name=theme-color content="#2e2e33"><meta name=msapplication-TileColor content="#2e2e33"><link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;0,700;1,400&&family=Noto+Serif&family=Playfair+Display:wght@500&display=swap" rel=stylesheet><meta name=generator content="Hugo 0.111.3"><noscript><style>#theme-toggle,.top-link{display:none}</style><style>@media(prefers-color-scheme:dark){:root{--theme:rgb(29, 30, 32);--entry:rgb(46, 46, 51);--primary:rgb(218, 218, 219);--secondary:rgb(155, 156, 157);--tertiary:rgb(65, 66, 68);--content:rgb(196, 196, 197);--hljs-bg:rgb(46, 46, 51);--code-bg:rgb(55, 56, 62);--border:rgb(51, 51, 51)}.list{background:var(--theme)}.list:not(.dark)::-webkit-scrollbar-track{background:0 0}.list:not(.dark)::-webkit-scrollbar-thumb{border-color:var(--theme)}}</style></noscript><meta property="og:title" content="Issue 15"><meta property="og:description" content="This week there&rsquo;s some more thoughts on AlphaFold and intrinsic disorder, a new experimental approach to look at high-energy protein conformations, and a structural angle to population genetics"><meta property="og:type" content="article"><meta property="og:url" content="https://folded-weekly.netlify.app/posts/15/"><meta property="og:image" content="https://folded-weekly.netlify.app/img/15.png"><meta property="article:section" content="posts"><meta property="article:published_time" content="2022-03-15T23:00:08+01:00"><meta property="article:modified_time" content="2022-03-15T23:00:08+01:00"><meta property="og:site_name" content="Folded Weekly"><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://folded-weekly.netlify.app/img/15.png"><meta name=twitter:title content="Issue 15"><meta name=twitter:description content="This week there&rsquo;s some more thoughts on AlphaFold and intrinsic disorder, a new experimental approach to look at high-energy protein conformations, and a structural angle to population genetics"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Posts","item":"https://folded-weekly.netlify.app/posts/"},{"@type":"ListItem","position":2,"name":"Issue 15","item":"https://folded-weekly.netlify.app/posts/15/"}]}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","headline":"Issue 15","name":"Issue 15","description":"This week there\u0026rsquo;s some more thoughts on AlphaFold and intrinsic disorder, a new experimental approach to look at high-energy protein conformations, and a structural angle to population genetics","keywords":[],"articleBody":"Intrinsic Protein Disorder, Conditional Folding and AlphaFold2 Since AlphaFold2 “won” the CASP competition in 2020, intrinsically disordered regions (IDRs) have been one of the points of discussion when it comes to AlphaFold2 performance. When the method was published, it was noted that the local predicted LDDT (pLDDT) score correlated well with IDRs. In this short communication, the authors explore this concept further and experiment with other metrics, such as relative solvent accessibility, in order to benchmark IDR prediction by AlphaFold2. Additionally, they also evaluate how pLDDT and relative solvent accessibility can be used to identify regions that fold conditionally upon binding. The conclusion is that AlphaFold2 performs well in the prediction of IDRs but is two orders of magnitude slower and, thus, not the best choice for fast searches.\nStructure determination of high-energy states in a dynamic protein ensemble Like missing those couple of seconds giving out clues about who killed Mr. Thrombey, transitory high-energy conformations can contain crucial information about the mechanism of actions of proteins. In this paper, the authors describe a way to access them using paramagnetic NMR experiments, through the use of pseudocontact shifts (PCS). These PCS provide long-distance structural information that can be used as restraints to determine conformations using simulated annealing, starting from known structures. This method was applied to the adenylate kinase (Adk) metalloprotein enzyme for which only the closed and open conformations are known, and confirmed several observations made by FRET or meta-dynamics simulations. They also show that the method can be generalized to non-metalloproteins by using a metal-binding tag.\nStructure-informed microbial population genetics elucidate selective pressures that shape protein evolution A novel use-case for structure prediction, microbial population genetics. The authors look into evolutionary dynamics in a popular ocean surface microbe taxon through the lens of protein structure as predicted by AlphaFold2. For this, they calculate the rate of synonymous and non-synonymous mutations at each single codon variant site and compare this to relative solvent accessibility and distance from the (predicted) ligand site for the amino acid under consideration. Overall, synonymous variants were way more common than non-synonymous, but the rates of non-synonymous mutations displayed some pretty interesting trends. They were a lot more common in exposed sites vs. buried sites, and they tended to avoid active sites and ligand-binding sites, the first likely due to maintaining structural stability and the second to maintain function. For one protein involved in recycling nitrogen, correctly taking into account its quaternary structure brought forth the expected conclusion that non-synonymous variation again avoids buried and active sites in this protein. This, combined with possible errors in the prediction of ligand binding sites and the steric configurations of residues, means that care needs to be taken in such large-scale structural analyses - though the kinds of conclusions one can draw from them are definitely unique and complementary to the common nucleotide sequence-based analysis.\nFigure of the Week Fig. 3: Linkers L1 and L2 mediate the structural transition to the active state.\na, Overview of the 18–20 MM active conformation. b, c, Detailed view of HNH (b) and RuvC (c) active sites. d, Docking of the L1 linker helix against the PAM-distal gRNA–TS duplex, shown as an isosurface representation. e, Interactions of L1 and L2 regions with the minor groove of the gRNA–TS duplex. HNH extending from L1 and L2 linkers has been removed for clarity and does not interact with this region of the gRNA–TS duplex.\nSource: Structural basis for mismatch surveillance by CRISPR–Cas9\nSome more FastFold: Reducing AlphaFold Training Time from 11 Days to 67 Hours Jalview with a pretty sweet update Jalview 2.11.2.0 is out!\nThis 'minor release' adds\n- @UCSFChimeraX, #Pymol (ce/incentive)\n- discovery of structures from 3D-Beacons @emblebi including #alphafold and @SWISS_MODEL - a new command line tool\nDowload \u0026 release notes via https://t.co/hMw7J57lAF pic.twitter.com/hXAiXpERLV\n— Jalview.org (@Jalview) March 10, 2022 MolEvolvR: A web-app for characterizing proteins using molecular evolution and phylogeny What have we learned from design of function in large proteins? CoDNaS-Q: a database of conformational diversity of the native state of proteins with quaternary structure Structural basis for mismatch surveillance by CRISPR–Cas9 How does a small molecule bind at a cryptic binding site? AtomNet PoseRanker: Enriching Ligand Pose Quality for Dynamic Proteins in Virtual High-Throughput Screens A new OpenMM Jupyter notebook We've added a @ProjectJupyter notebook illustrating how to use openmm-torch to run a GPU-accelerated simulation of alanine dipeptide using the ANI-2x ML potential from @adrian_roitberg @olexandr @gpusciguy within OpenMM!\nCheck it out on @GooogleColab:https://t.co/IwrCLPXVNF\n— OpenMM (@openmm_toolkit) March 4, 2022 Residue Assignment in Crystallographic Protein Electron Density Maps with 3D Convolutional Networks The Carbon Footprint of Bioinformatics BINANA 2: Characterizing Receptor/Ligand Interactions in Python and JavaScript Sampling alternative conformational states of transporters and receptors with AlphaFold2 Protein-Ligand Interaction Graphs: Learning from Ligand-Shaped 3D Interaction Graphs to Improve Binding Affinity Prediction Accelerating therapeutic protein design On the Value of Using 3D Shape and Electrostatic Similarities in Deep Generative Methods An update of AlphaFold-Multimer that helps with the steric clash issues that were cropping up We have updated the AlphaFold-Multimer paper and code with a new set of protein complex models trained with an improved loss. These models reduce the number of steric clashes and improve accuracy.\nCode: https://t.co/LgqAo6GKzc\nPaper: https://t.co/A3uTYVoTeq pic.twitter.com/SHKZzhAX82\n— Tim Green (@tfgg2) March 10, 2022 ","wordCount":"879","inLanguage":"en","image":"https://folded-weekly.netlify.app/img/15.png","datePublished":"2022-03-15T23:00:08+01:00","dateModified":"2022-03-15T23:00:08+01:00","mainEntityOfPage":{"@type":"WebPage","@id":"https://folded-weekly.netlify.app/posts/15/"},"publisher":{"@type":"Organization","name":"Folded Weekly","logo":{"@type":"ImageObject","url":"https://folded-weekly.netlify.app/favicon.ico"}}}</script></head><body id=top><script>localStorage.getItem("pref-theme")==="dark"?document.body.classList.add("dark"):localStorage.getItem("pref-theme")==="light"?document.body.classList.remove("dark"):window.matchMedia("(prefers-color-scheme: dark)").matches&&document.body.classList.add("dark")</script><header class=header><nav class=nav><div class=logo><a href=https://folded-weekly.netlify.app/ accesskey=h title="Home (Alt + H)"><img src=https://folded-weekly.netlify.app/logo_full.png alt=logo aria-label=logo height=200>Home</a>
<span class=logo-switches><button id=theme-toggle accesskey=t title="(Alt + T)"><svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21 12.79A9 9 0 1111.21 3 7 7 0 0021 12.79z"/></svg><svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><circle cx="12" cy="12" r="5"/><line x1="12" y1="1" x2="12" y2="3"/><line x1="12" y1="21" x2="12" y2="23"/><line x1="4.22" y1="4.22" x2="5.64" y2="5.64"/><line x1="18.36" y1="18.36" x2="19.78" y2="19.78"/><line x1="1" y1="12" x2="3" y2="12"/><line x1="21" y1="12" x2="23" y2="12"/><line x1="4.22" y1="19.78" x2="5.64" y2="18.36"/><line x1="18.36" y1="5.64" x2="19.78" y2="4.22"/></svg></button></span></div><ul id=menu><li><a href=https://folded-weekly.netlify.app/search/ title="Search (Alt + /)" accesskey=/><span>Search</span></a></li><li><a href=https://folded-weekly.netlify.app/archives title=Archive><span>Archive</span></a></li></ul></nav></header><main class=main><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css integrity=sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3+Aro6EYUG4+cU+KJWu/X crossorigin=anonymous><script defer src=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js integrity=sha384-g7c+Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI+sEnkvrMWph2EDg4 crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js integrity=sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC+Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa crossorigin=anonymous onload=renderMathInElement(document.body)></script><article class=post-single><header class=post-header><h1 class=post-title><figure class=entry-cover><img loading=lazy src=https://folded-weekly.netlify.app/img/15.png alt=15></figure>Issue 15</h1><div class=post-meta><span title='2022-03-15 23:00:08 +0100 CET'>March 15, 2022</span>&nbsp;·&nbsp;5 min</div></header><div class=toc><details open><summary accesskey=c title="(Alt + C)"><span class=details>Table of Contents</span></summary><div class=inner><ul><li><a href=#intrinsic-protein-disorder-conditional-folding-and-alphafold2httpsdoiorg10110120220303482768 aria-label="Intrinsic Protein Disorder, Conditional Folding and AlphaFold2">Intrinsic Protein Disorder, Conditional Folding and AlphaFold2</a></li><li><a href=#structure-determination-of-high-energy-states-in-a-dynamic-protein-ensemblehttpsdoiorg101038s41586-022-04468-9 aria-label="Structure determination of high-energy states in a dynamic protein ensemble">Structure determination of high-energy states in a dynamic protein ensemble</a></li><li><a href=#structure-informed-microbial-population-genetics-elucidate-selective-pressures-that-shape-protein-evolutionhttpsdoiorg10110120220302482602 aria-label="Structure-informed microbial population genetics elucidate selective pressures that shape protein evolution">Structure-informed microbial population genetics elucidate selective pressures that shape protein evolution</a></li><li><a href=#figure-of-the-week aria-label="Figure of the Week">Figure of the Week</a></li><li><a href=#some-more aria-label="Some more">Some more</a></li></ul></div></details></div><div class=post-content><h2 id=intrinsic-protein-disorder-conditional-folding-and-alphafold2httpsdoiorg10110120220303482768><a href=https://doi.org/10.1101/2022.03.03.482768>Intrinsic Protein Disorder, Conditional Folding and AlphaFold2</a><a hidden class=anchor aria-hidden=true href=#intrinsic-protein-disorder-conditional-folding-and-alphafold2httpsdoiorg10110120220303482768>#</a></h2><p>Since AlphaFold2 &ldquo;won&rdquo; the CASP competition in 2020, intrinsically disordered regions (IDRs) have been one of the points of discussion when it comes to AlphaFold2 performance. When the method was published, it was noted that the local predicted LDDT (pLDDT) score correlated well with IDRs. In this short communication, the authors explore this concept further and experiment with other metrics, such as relative solvent accessibility, in order to benchmark IDR prediction by AlphaFold2. Additionally, they also evaluate how pLDDT and relative solvent accessibility can be used to identify regions that fold conditionally upon binding. The conclusion is that AlphaFold2 performs well in the prediction of IDRs but is two orders of magnitude slower and, thus, not the best choice for fast searches.</p><h2 id=structure-determination-of-high-energy-states-in-a-dynamic-protein-ensemblehttpsdoiorg101038s41586-022-04468-9><a href=https://doi.org/10.1038/s41586-022-04468-9>Structure determination of high-energy states in a dynamic protein ensemble</a><a hidden class=anchor aria-hidden=true href=#structure-determination-of-high-energy-states-in-a-dynamic-protein-ensemblehttpsdoiorg101038s41586-022-04468-9>#</a></h2><p>Like missing those couple of seconds giving out clues about who killed Mr. Thrombey, transitory high-energy conformations can contain crucial information about the mechanism of actions of proteins. In this paper, the authors describe a way to access them using paramagnetic NMR experiments, through the use of pseudocontact shifts (PCS). These PCS provide long-distance structural information that can be used as restraints to determine conformations using simulated annealing, starting from known structures. This method was applied to the adenylate kinase (Adk) metalloprotein enzyme for which only the closed and open conformations are known, and confirmed several observations made by FRET or meta-dynamics simulations. They also show that the method can be generalized to non-metalloproteins by using a metal-binding tag.</p><h2 id=structure-informed-microbial-population-genetics-elucidate-selective-pressures-that-shape-protein-evolutionhttpsdoiorg10110120220302482602><a href=https://doi.org/10.1101/2022.03.02.482602>Structure-informed microbial population genetics elucidate selective pressures that shape protein evolution</a><a hidden class=anchor aria-hidden=true href=#structure-informed-microbial-population-genetics-elucidate-selective-pressures-that-shape-protein-evolutionhttpsdoiorg10110120220302482602>#</a></h2><p>A novel use-case for structure prediction, microbial population genetics. The authors look into evolutionary dynamics in a popular ocean surface microbe taxon through the lens of protein structure as predicted by AlphaFold2. For this, they calculate the rate of synonymous and non-synonymous mutations at each single codon variant site and compare this to relative solvent accessibility and distance from the (predicted) ligand site for the amino acid under consideration. Overall, synonymous variants were way more common than non-synonymous, but the rates of non-synonymous mutations displayed some pretty interesting trends. They were a lot more common in exposed sites vs. buried sites, and they tended to avoid active sites and ligand-binding sites, the first likely due to maintaining structural stability and the second to maintain function. For one protein involved in recycling nitrogen, correctly taking into account its quaternary structure brought forth the expected conclusion that non-synonymous variation again avoids buried and active sites in this protein. This, combined with possible errors in the prediction of ligand binding sites and the steric configurations of residues, means that care needs to be taken in such large-scale structural analyses - though the kinds of conclusions one can draw from them are definitely unique and complementary to the common nucleotide sequence-based analysis.</p><h2 id=figure-of-the-week>Figure of the Week<a hidden class=anchor aria-hidden=true href=#figure-of-the-week>#</a></h2><figure><img loading=lazy src=https://folded-weekly.netlify.app/img/15_fotw.png></figure><blockquote><p>Fig. 3: Linkers L1 and L2 mediate the structural transition to the active state.</p><p><strong>a</strong>, Overview of the 18–20 MM active conformation. <strong>b</strong>, <strong>c</strong>, Detailed view of HNH (<strong>b</strong>) and RuvC (<strong>c</strong>) active sites. <strong>d</strong>, Docking of the L1 linker helix against the PAM-distal gRNA–TS duplex, shown as an isosurface representation. <strong>e</strong>, Interactions of L1 and L2 regions with the minor groove of the gRNA–TS duplex. HNH extending from L1 and L2 linkers has been removed for clarity and does not interact with this region of the gRNA–TS duplex.</p></blockquote><p><strong>Source: <a href=https://doi.org/10.1038/s41586-022-04470-1>Structural basis for mismatch surveillance by CRISPR–Cas9</a></strong></p><h2 id=some-more>Some more<a hidden class=anchor aria-hidden=true href=#some-more>#</a></h2><ul><li><a href=https://doi.org/10.48550/arXiv.2203.00854>FastFold: Reducing AlphaFold Training Time from 11 Days to 67 Hours</a></li><li>Jalview with a pretty sweet update<blockquote class=twitter-tweet><p lang=en dir=ltr>Jalview 2.11.2.0 is out!<br><br>This 'minor release' adds<br>- <a href="https://twitter.com/UCSFChimeraX?ref_src=twsrc%5Etfw">@UCSFChimeraX</a>, <a href="https://twitter.com/hashtag/Pymol?src=hash&amp;ref_src=twsrc%5Etfw">#Pymol</a> (ce/incentive)<br>- discovery of structures from 3D-Beacons <a href="https://twitter.com/emblebi?ref_src=twsrc%5Etfw">@emblebi</a> including <a href="https://twitter.com/hashtag/alphafold?src=hash&amp;ref_src=twsrc%5Etfw">#alphafold</a> and <a href="https://twitter.com/SWISS_MODEL?ref_src=twsrc%5Etfw">@SWISS_MODEL</a><br>- a new command line tool<br><br>Dowload & release notes via <a href=https://t.co/hMw7J57lAF>https://t.co/hMw7J57lAF</a> <a href=https://t.co/hXAiXpERLV>pic.twitter.com/hXAiXpERLV</a></p>&mdash; Jalview.org (@Jalview) <a href="https://twitter.com/Jalview/status/1501874697826447361?ref_src=twsrc%5Etfw">March 10, 2022</a></blockquote><script async src=https://platform.twitter.com/widgets.js></script></li><li><a href=https://doi.org/10.1101/2022.02.18.461833>MolEvolvR: A web-app for characterizing proteins using molecular evolution and phylogeny</a></li><li><a href=https://downloads.spj.sciencemag.org/bdr/2022/9787581.pdf>What have we learned from design of function in large proteins?</a></li><li><a href=https://doi.org/10.1101/2022.02.15.480082>CoDNaS-Q: a database of conformational diversity of the native state of proteins with quaternary structure</a></li><li><a href=https://doi.org/10.1038/s41586-022-04470-1>Structural basis for mismatch surveillance by CRISPR–Cas9</a></li><li><a href=https://doi.org/10.1371/journal.pcbi.1009817>How does a small molecule bind at a cryptic binding site?</a></li><li><a href=https://doi.org/10.1021/acs.jcim.1c01250>AtomNet PoseRanker: Enriching Ligand Pose Quality for Dynamic Proteins in Virtual High-Throughput Screens</a></li><li>A new OpenMM Jupyter notebook<blockquote class=twitter-tweet><p lang=en dir=ltr>We've added a <a href="https://twitter.com/ProjectJupyter?ref_src=twsrc%5Etfw">@ProjectJupyter</a> notebook illustrating how to use openmm-torch to run a GPU-accelerated simulation of alanine dipeptide using the ANI-2x ML potential from <a href="https://twitter.com/adrian_roitberg?ref_src=twsrc%5Etfw">@adrian_roitberg</a> <a href="https://twitter.com/olexandr?ref_src=twsrc%5Etfw">@olexandr</a> <a href="https://twitter.com/gpusciguy?ref_src=twsrc%5Etfw">@gpusciguy</a> within OpenMM!<br><br>Check it out on @GooogleColab:<a href=https://t.co/IwrCLPXVNF>https://t.co/IwrCLPXVNF</a></p>&mdash; OpenMM (@openmm_toolkit) <a href="https://twitter.com/openmm_toolkit/status/1499738933529235456?ref_src=twsrc%5Etfw">March 4, 2022</a></blockquote><script async src=https://platform.twitter.com/widgets.js></script></li><li><a href=https://doi.org/10.1109/ACCESS.2022.3156108>Residue Assignment in Crystallographic Protein Electron Density Maps with 3D Convolutional Networks</a></li><li><a href=https://doi.org/10.1093/molbev/msac034>The Carbon Footprint of Bioinformatics</a></li><li><a href=https://doi.org/10.1021/acs.jcim.1c01461>BINANA 2: Characterizing Receptor/Ligand Interactions in Python and JavaScript</a></li><li><a href=https://doi.org/10.7554/eLife.75751>Sampling alternative conformational states of transporters and receptors with AlphaFold2</a></li><li><a href=https://doi.org/10.1101/2022.03.04.483012>Protein-Ligand Interaction Graphs: Learning from Ligand-Shaped 3D Interaction Graphs to Improve Binding Affinity Prediction</a></li><li><a href=https://doi.org/10.1016/bs.apcsb.2022.01.004>Accelerating therapeutic protein design</a></li><li><a href=https://doi.org/10.1021/acs.jcim.1c01535>On the Value of Using 3D Shape and Electrostatic Similarities in Deep Generative Methods</a></li><li>An update of AlphaFold-Multimer that helps with the steric clash issues that were cropping up<blockquote class=twitter-tweet><p lang=en dir=ltr>We have updated the AlphaFold-Multimer paper and code with a new set of protein complex models trained with an improved loss. These models reduce the number of steric clashes and improve accuracy.<br><br>Code: <a href=https://t.co/LgqAo6GKzc>https://t.co/LgqAo6GKzc</a><br>Paper: <a href=https://t.co/A3uTYVoTeq>https://t.co/A3uTYVoTeq</a> <a href=https://t.co/SHKZzhAX82>pic.twitter.com/SHKZzhAX82</a></p>&mdash; Tim Green (@tfgg2) <a href="https://twitter.com/tfgg2/status/1501945723620769793?ref_src=twsrc%5Etfw">March 10, 2022</a></blockquote><script async src=https://platform.twitter.com/widgets.js></script></li></ul></div><div class=subscribe><form action=https://buttondown.email/api/emails/embed-subscribe/FoldedWeekly method=post target=popupwindow onsubmit='window.open("https://buttondown.email/FoldedWeekly","popupwindow")' class=embeddable-buttondown-form><input type=email name=email id=bd-email placeholder="Enter your email">
<input type=submit value=Subscribe><p><a href=https://buttondown.email target=_blank id=bd-link>Powered by Buttondown</a></p></form></div><footer class=post-footer><nav class=paginav><a class=prev href=https://folded-weekly.netlify.app/posts/16/><span class=title>« Prev Page</span><br><span>Issue 16</span></a>
<a class=next href=https://folded-weekly.netlify.app/posts/14/><span class=title>Next Page »</span><br><span>Issue 14</span></a></nav><div class=share-buttons><a target=_blank rel="noopener noreferrer" aria-label="share Issue 15 on twitter" href="https://twitter.com/intent/tweet/?text=Issue%2015&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f15%2f&amp;hashtags="><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM195.519 424.544c135.939.0 210.268-112.643 210.268-210.268.0-3.218.0-6.437-.153-9.502 14.406-10.421 26.973-23.448 36.935-38.314-13.18 5.824-27.433 9.809-42.452 11.648 15.326-9.196 26.973-23.602 32.49-40.92-14.252 8.429-30.038 14.56-46.896 17.931-13.487-14.406-32.644-23.295-53.946-23.295-40.767.0-73.87 33.104-73.87 73.87.0 5.824.613 11.494 1.992 16.858-61.456-3.065-115.862-32.49-152.337-77.241-6.284 10.881-9.962 23.601-9.962 37.088.0 25.594 13.027 48.276 32.95 61.456-12.107-.307-23.448-3.678-33.41-9.196v.92c0 35.862 25.441 65.594 59.311 72.49-6.13 1.686-12.72 2.606-19.464 2.606-4.751.0-9.348-.46-13.946-1.38 9.349 29.426 36.628 50.728 68.965 51.341-25.287 19.771-57.164 31.571-91.8 31.571-5.977.0-11.801-.306-17.625-1.073 32.337 21.15 71.264 33.41 112.95 33.41z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 15 on linkedin" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f15%2f&amp;title=Issue%2015&amp;summary=Issue%2015&amp;source=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f15%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM160.461 423.278V197.561h-75.04v225.717h75.04zm270.539.0V293.839c0-69.333-37.018-101.586-86.381-101.586-39.804.0-57.634 21.891-67.617 37.266v-31.958h-75.021c.995 21.181.0 225.717.0 225.717h75.02V297.222c0-6.748.486-13.492 2.474-18.315 5.414-13.475 17.767-27.434 38.494-27.434 27.135.0 38.007 20.707 38.007 51.037v120.768H431zM123.448 88.722C97.774 88.722 81 105.601 81 127.724c0 21.658 16.264 39.002 41.455 39.002h.484c26.165.0 42.452-17.344 42.452-39.002-.485-22.092-16.241-38.954-41.943-39.002z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 15 on reddit" href="https://reddit.com/submit?url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f15%2f&title=Issue%2015"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM446 265.638c0-22.964-18.616-41.58-41.58-41.58-11.211.0-21.361 4.457-28.841 11.666-28.424-20.508-67.586-33.757-111.204-35.278l18.941-89.121 61.884 13.157c.756 15.734 13.642 28.29 29.56 28.29 16.407.0 29.706-13.299 29.706-29.701.0-16.403-13.299-29.702-29.706-29.702-11.666.0-21.657 6.792-26.515 16.578l-69.105-14.69c-1.922-.418-3.939-.042-5.585 1.036-1.658 1.073-2.811 2.761-3.224 4.686l-21.152 99.438c-44.258 1.228-84.046 14.494-112.837 35.232-7.468-7.164-17.589-11.591-28.757-11.591-22.965.0-41.585 18.616-41.585 41.58.0 16.896 10.095 31.41 24.568 37.918-.639 4.135-.99 8.328-.99 12.576.0 63.977 74.469 115.836 166.33 115.836s166.334-51.859 166.334-115.836c0-4.218-.347-8.387-.977-12.493 14.564-6.47 24.735-21.034 24.735-38.001zM326.526 373.831c-20.27 20.241-59.115 21.816-70.534 21.816-11.428.0-50.277-1.575-70.522-21.82-3.007-3.008-3.007-7.882.0-10.889 3.003-2.999 7.882-3.003 10.885.0 12.777 12.781 40.11 17.317 59.637 17.317 19.522.0 46.86-4.536 59.657-17.321 3.016-2.999 7.886-2.995 10.885.008 3.008 3.011 3.003 7.882-.008 10.889zm-5.23-48.781c-16.373.0-29.701-13.324-29.701-29.698.0-16.381 13.328-29.714 29.701-29.714 16.378.0 29.706 13.333 29.706 29.714.0 16.374-13.328 29.698-29.706 29.698zM160.91 295.348c0-16.381 13.328-29.71 29.714-29.71 16.369.0 29.689 13.329 29.689 29.71.0 16.373-13.32 29.693-29.689 29.693-16.386.0-29.714-13.32-29.714-29.693z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 15 on facebook" href="https://facebook.com/sharer/sharer.php?u=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f15%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H342.978V319.085h66.6l12.672-82.621h-79.272v-53.617c0-22.603 11.073-44.636 46.58-44.636H425.6v-70.34s-32.71-5.582-63.982-5.582c-65.288.0-107.96 39.569-107.96 111.204v62.971h-72.573v82.621h72.573V512h-191.104c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 15 on whatsapp" href="https://api.whatsapp.com/send?text=Issue%2015%20-%20https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f15%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zm-58.673 127.703c-33.842-33.881-78.847-52.548-126.798-52.568-98.799.0-179.21 80.405-179.249 179.234-.013 31.593 8.241 62.428 23.927 89.612l-25.429 92.884 95.021-24.925c26.181 14.28 55.659 21.807 85.658 21.816h.074c98.789.0 179.206-80.413 179.247-179.243.018-47.895-18.61-92.93-52.451-126.81zM263.976 403.485h-.06c-26.734-.01-52.954-7.193-75.828-20.767l-5.441-3.229-56.386 14.792 15.05-54.977-3.542-5.637c-14.913-23.72-22.791-51.136-22.779-79.287.033-82.142 66.867-148.971 149.046-148.971 39.793.014 77.199 15.531 105.329 43.692 28.128 28.16 43.609 65.592 43.594 105.4-.034 82.149-66.866 148.983-148.983 148.984zm81.721-111.581c-4.479-2.242-26.499-13.075-30.604-14.571-4.105-1.495-7.091-2.241-10.077 2.241-2.986 4.483-11.569 14.572-14.182 17.562-2.612 2.988-5.225 3.364-9.703 1.12-4.479-2.241-18.91-6.97-36.017-22.23C231.8 264.15 222.81 249.484 220.198 245s-.279-6.908 1.963-9.14c2.016-2.007 4.48-5.232 6.719-7.847 2.24-2.615 2.986-4.484 4.479-7.472 1.493-2.99.747-5.604-.374-7.846-1.119-2.241-10.077-24.288-13.809-33.256-3.635-8.733-7.327-7.55-10.077-7.688-2.609-.13-5.598-.158-8.583-.158-2.986.0-7.839 1.121-11.944 5.604-4.105 4.484-15.675 15.32-15.675 37.364.0 22.046 16.048 43.342 18.287 46.332 2.24 2.99 31.582 48.227 76.511 67.627 10.685 4.615 19.028 7.371 25.533 9.434 10.728 3.41 20.492 2.929 28.209 1.775 8.605-1.285 26.499-10.833 30.231-21.295 3.732-10.464 3.732-19.431 2.612-21.298-1.119-1.869-4.105-2.99-8.583-5.232z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 15 on telegram" href="https://telegram.me/share/url?text=Issue%2015&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f15%2f"><svg viewBox="2 2 28 28"><path d="M26.49 29.86H5.5a3.37 3.37.0 01-2.47-1 3.35 3.35.0 01-1-2.47V5.48A3.36 3.36.0 013 3 3.37 3.37.0 015.5 2h21A3.38 3.38.0 0129 3a3.36 3.36.0 011 2.46V26.37a3.35 3.35.0 01-1 2.47 3.38 3.38.0 01-2.51 1.02zm-5.38-6.71a.79.79.0 00.85-.66L24.73 9.24a.55.55.0 00-.18-.46.62.62.0 00-.41-.17q-.08.0-16.53 6.11a.59.59.0 00-.41.59.57.57.0 00.43.52l4 1.24 1.61 4.83a.62.62.0 00.63.43.56.56.0 00.4-.17L16.54 20l4.09 3A.9.9.0 0021.11 23.15zM13.8 20.71l-1.21-4q8.72-5.55 8.78-5.55c.15.0.23.0.23.16a.18.18.0 010 .06s-2.51 2.3-7.52 6.8z"/></svg></a></div></footer></article></main><footer class=footer><span>&copy; 2023 <a href=https://folded-weekly.netlify.app/>Folded Weekly</a></span>
<span>Powered by
<a href=https://gohugo.io/ rel="noopener noreferrer" target=_blank>Hugo</a> &
        <a href=https://git.io/hugopapermod rel=noopener target=_blank>PaperMod</a></span></footer><a href=#top aria-label="go to top" title="Go to Top (Alt + G)" class=top-link id=top-link accesskey=g><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentcolor"><path d="M12 6H0l6-6z"/></svg></a><script>let menu=document.getElementById("menu");menu&&(menu.scrollLeft=localStorage.getItem("menu-scroll-position"),menu.onscroll=function(){localStorage.setItem("menu-scroll-position",menu.scrollLeft)}),document.querySelectorAll('a[href^="#"]').forEach(e=>{e.addEventListener("click",function(e){e.preventDefault();var t=this.getAttribute("href").substr(1);window.matchMedia("(prefers-reduced-motion: reduce)").matches?document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView():document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView({behavior:"smooth"}),t==="top"?history.replaceState(null,null," "):history.pushState(null,null,`#${t}`)})})</script><script>var mybutton=document.getElementById("top-link");window.onscroll=function(){document.body.scrollTop>800||document.documentElement.scrollTop>800?(mybutton.style.visibility="visible",mybutton.style.opacity="1"):(mybutton.style.visibility="hidden",mybutton.style.opacity="0")}</script><script>document.getElementById("theme-toggle").addEventListener("click",()=>{document.body.className.includes("dark")?(document.body.classList.remove("dark"),localStorage.setItem("pref-theme","light")):(document.body.classList.add("dark"),localStorage.setItem("pref-theme","dark"))})</script></body></html>