<!doctype html><html lang=en dir=auto><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=robots content="index, follow"><title>Issue 14 | Folded Weekly</title><meta name=keywords content><meta name=description content="This week we cover protein annotation, evolution, representation, mutation, and interaction!"><meta name=author content><link rel=canonical href=https://folded-weekly.netlify.app/posts/14/><link crossorigin=anonymous href=/assets/css/stylesheet.min.0caf3925a7f1eae074a9aadc5d80291998cf748272cb5b0e65ca8199d38d7909.css integrity="sha256-DK85Jafx6uB0qarcXYApGZjPdIJyy1sOZcqBmdONeQk=" rel="preload stylesheet" as=style><link rel=preload href=logo_full.png as=image><script defer crossorigin=anonymous src=/assets/js/highlight.min.b95bacdc39e37a332a9f883b1e78be4abc1fdca2bc1f2641f55e3cd3dabd4d61.js integrity="sha256-uVus3DnjejMqn4g7Hni+Srwf3KK8HyZB9V4809q9TWE=" onload=hljs.initHighlightingOnLoad()></script>
<link rel=icon href=https://folded-weekly.netlify.app/favicon.ico><link rel=icon type=image/png sizes=16x16 href=https://folded-weekly.netlify.app/favicon-16x16.png><link rel=icon type=image/png sizes=32x32 href=https://folded-weekly.netlify.app/favicon-32x32.png><link rel=apple-touch-icon href=https://folded-weekly.netlify.app/apple-touch-icon.png><link rel=mask-icon href=https://folded-weekly.netlify.app/favicon-16x16.png><meta name=theme-color content="#2e2e33"><meta name=msapplication-TileColor content="#2e2e33"><link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;0,700;1,400&&family=Noto+Serif&family=Playfair+Display:wght@500&display=swap" rel=stylesheet><meta name=generator content="Hugo 0.111.3"><noscript><style>#theme-toggle,.top-link{display:none}</style><style>@media(prefers-color-scheme:dark){:root{--theme:rgb(29, 30, 32);--entry:rgb(46, 46, 51);--primary:rgb(218, 218, 219);--secondary:rgb(155, 156, 157);--tertiary:rgb(65, 66, 68);--content:rgb(196, 196, 197);--hljs-bg:rgb(46, 46, 51);--code-bg:rgb(55, 56, 62);--border:rgb(51, 51, 51)}.list{background:var(--theme)}.list:not(.dark)::-webkit-scrollbar-track{background:0 0}.list:not(.dark)::-webkit-scrollbar-thumb{border-color:var(--theme)}}</style></noscript><meta property="og:title" content="Issue 14"><meta property="og:description" content="This week we cover protein annotation, evolution, representation, mutation, and interaction!"><meta property="og:type" content="article"><meta property="og:url" content="https://folded-weekly.netlify.app/posts/14/"><meta property="og:image" content="https://folded-weekly.netlify.app/img/14.png"><meta property="article:section" content="posts"><meta property="article:published_time" content="2022-03-03T14:35:29+01:00"><meta property="article:modified_time" content="2022-03-03T14:35:29+01:00"><meta property="og:site_name" content="Folded Weekly"><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://folded-weekly.netlify.app/img/14.png"><meta name=twitter:title content="Issue 14"><meta name=twitter:description content="This week we cover protein annotation, evolution, representation, mutation, and interaction!"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Posts","item":"https://folded-weekly.netlify.app/posts/"},{"@type":"ListItem","position":2,"name":"Issue 14","item":"https://folded-weekly.netlify.app/posts/14/"}]}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","headline":"Issue 14","name":"Issue 14","description":"This week we cover protein annotation, evolution, representation, mutation, and interaction!","keywords":[],"articleBody":"A double issue covering the last two weeks of updates in structural bioinformatics\nUsing deep learning to annotate the protein universe Each year the wealth of protein-coding sequences uncovered by multiple genomic projects increases exponentially. This makes their annotation for functional features challenging but also more and more important. Metagenomic projects are an especially big source of novel protein sequences and while many for sure represent families completly new to us, a much bigger fraction most likely correspond to highly divergent forms of well known families that just escape detection by HMM-based annotation methods. In this work, the authors trained a network on the domain families in Pfam and demonstrate how deep learning allows the annotation and classification of sequences seemingly without clear homologs, expanding the annotation coverage of the protein universe. The method is useful for the annotation of single domain sequences and allowed for the merging of DUF families with others well known. It does not, however, map annotations into the full-length sequence. The authors circumvent the annotation of multi-domain sequences without known domain boundaries by using a sliding window. The code is freely available and a Colab notebook makes it really convenient to play with your favourite protein. One word of caution however: while powerful and indeed superior in sensitivity to HMMs, the network always provides a result; it always finds the closest annotation in embeded space for a given sequence, even if it is really far away.\nIn silico evolution of nucleic acid-binding proteins from a nonfunctional scaffold The authors tackle the fascinating problem of designing proteins with a given function, using simulated directed evolution. To make the task even more challenging they demonstrate their approach, Proseeker, on a completely de novo designed repeat protein with no existing function, designed just to optimize its consecutive alpha-helix folding capabilities. To imbue this protein with nucleotide-binding functionality, the authors create an assessment library of phycochemical descriptors describing short peptides from pentatricopeptide repeat proteins, specifically those covering the RNA-binding residues. These descriptors are clustered and any newly evolved protein is compared to the representative descriptors from each cluster to determine its fitness. Two residues are mutated at each step in the process, with residues around the first mutation site getting a small boost in being selected as the second mutation site, so as to boost creation of mutational islands and epistatic interactions. Then, selection is done computationally using DP-Bind - an SVM trained to predict DNA binding ability. The top scoring 10% variants are used to generate the next round of mutants. Proseeker is experimentally validated by producing five synthetically evolved proteins and running multiple in vitro and in vivo (with a pretty neat new design) assays, showing that three of them actually do have nucleotide-binding activity, an amazing result. Another nice result of the paper is describing all the different variables that are important for getting this kind of approach to work, and systematically varying each one to show its effect - of course this is only possible since the evolution is in silico!\nPyUUL provides an interface between biological structures and deep learning algorithms For the structural biologists who were not born with Machine Learning or fancy Neural Networks (NN), using these new methods can be sometimes intimidating. pyUUL is one of those tools that will help sand down some of the rough edges and make you finally jump on the bandwagon like the other cool kids. The aim of this package is to transform any 3D structures of your choice (protein, small molecules or nucleic acids) into a meaningful representations (voxel-based, surface or volumetric point-cloud) directly exploitable by any NN architecture. At the end, this idea is simple, but nonetheless very powerful. The authors also present multiple test cases, ranging from alpha helix identification, structure-based clustering and small-molecule optimization. Based on PyTorch, all the operations can be run on both CPU and GPU. The package is available here and more importantly it comes with documentation describing some of the examples presented in the papers.\nSystematic evaluation of computational tools to predict the effects of mutations on protein stability in the absence of experimental structures Methods for the prediction of protein stability upon mutation are usually evaluated and developed with experimental structures in mind. With the vast majority of structures for proteins being models and predictions, the authors tackled the question of how common mutation stability prediction tools would perform on protein structure models. They evaluated the performance of 14 protein stability predictors on a self-created data set of models based on a curated subset of structures in the ProTherm data set using MODELLER and AlphaFold2. The authors found that the protein stability predictions deteriorate for homology models with a target-template identity below 40%; especially for machine learning and structure based methods. The predictions on AlphaFold2 models were mostly consistent with predictions on experimental structures, which opens up stability predictions with the same accuracy as in experimental structures to all proteins. A limitation we see here is the entanglement of the evaluation of the models of proteins in the ProTherm data base, which is usually also used for the development of stability prediction methods.\nExploring protein-protein interactions at the proteome level The biological activity of proteins is defined by how they interact with other biological molecules, from small ligands, to nucleic acids, sugars and other proteins. Usually proteins do not act by themselves, and form strong or transient complexes with other proteins, building large macromolecular machines or transient interaction networks. It is thus essential to not only understand the three-dimensional structure of individual proteins but also in complex with their interaction partners. In this review, Elhabashy et al. discuss the different approaches available to analyse, predict and model such interactions at a large-scale. From experimental methods, to computational and hybrid approaches, they provide a concise but extremely informative and insightful state-of-the-art for the modelling of protein-protein interactions at the proteome level, also highlighting the recent role of machine learning in such tasks.\nFigure(s) of the Week(s) Figure 1. Molecular machinery of life Gallery of supramolecular complexes formed through protein-protein interactions. PDB IDs of shown structures from the left to right are as follows 1S5L: photosynthetic oxygen evolving center; 3J8K: tilted state of actin T2; 2GLS: glutamine synthetase; 3JA6: novel kinase conformational switch in bacterial chemotaxis signaling; 6R3Q: membrane adenylyl cyclase bound to an activated stimulatory G protein; 3JB9: yeast spliceosome; 1FNT: 20s proteasome from yeast in complex with the proteasome activator pa26 from Trypanosoma brucei; 3JBT: the Apaf-1 apoptosome; 7CGO: the flagellar motor-hook complex from Salmonella; 2GJX: human β-hexosaminidase A; 1I6H: RNA polymerase II elongation complex; 4V2T: membrane-embedded pleurotolysin pore with 13-fold symmetry; 5IDE: GluA2/3 AMPA receptor heterotetramer (model I); 1AV1: human apolipoprotein A-I; 4PJ1: human mitochondrial chaperonin symmetrical ‘football’ complex; 5LNK: ovine respiratory complex I; 6VQ6: mammalian V-ATPase from rat brain; 1IGT: immunoglobulin; 1BUW: s-nitroso-nitrosyl human hemoglobin A; 6ASY: human endoplasmic reticulum chaperone BiP; 3J9K: dark apoptosome in complex with the Dronc CARD domain; 4V60: rat liver half-vault; 3J9J: capsaicin receptor TRPV1; and 2GBL: circadian clock protein KaiC\nSource: Exploring protein-protein interactions at the proteome level\nFig. 3. Illustration of the LoopGrafter workflow. The server starts with the input of the PDB files or codes of two structures (orange): (i) the loop receptor (scaffold) and (ii) the loop donator (insert). The tool then calculates the secondary structures of all the loops, assesses the protein flexibility using normal mode analysis, and performs a superimposition of the two structures (blue). Once the user selects the loops of interest, the loops are paired. The suitable loop boundaries are then explored based on geometric restrictions so that maximal sequence diversity is probed in the designed grafted variants. The possible solutions are collected as a list of sequences, and the structures are then modeled and ranked by the respective stability scores obtained from Rosetta and MODELLER (yellow). The web service is provided free of charge for non-commercial use: https://loschmidt.chemi.muni.cz/loopgrafter/.\nSource: Tools for computational design and high-throughput screening of therapeutic enzymes\nMany more DisProt in 2022: improved quality and accessibility of protein intrinsic disorder annotation Accurate self-assessment in 2D: protein contact map quality estimation by deep evolutionary reconciliation Application of Homology Modeling by Enhanced Profile–Profile Alignment and Flexible-Fitting Simulation to Cryo-EM Based Structure Determination ProteinBERT: a universal deep-learning model of protein sequence and function MeCOM: A Method for Comparing Three-Dimensional Metalloenzyme Active Sites Model agnostic generation of counterfactual explanations for molecules Molecular modeling in drug discovery Ancestral reconstruction of duplicated signaling proteins reveals the evolution of signaling specificity Tools for computational design and high-throughput screening of therapeutic enzymes Visualizing protein breathing motions associated with aromatic ring flipping Forty years in cryoEM of membrane proteins The clinical variant interpretation portal DECIPHER now has AlphaFold structures: DECIPHER portal for clinical variant interpretation (free-to-use) now contains AlphaFold structures, annotated with known pathogenic variants, displayed interactively. https://t.co/rgbBXB5T4M pic.twitter.com/LLKmWs4Clf\n— Matt Hurles (@mehurles) February 18, 2022 Structural basis for SARS-CoV-2 Delta variant recognition of ACE2 receptor and broadly neutralizing antibodies SWAAT Bioinformatics Workflow for Protein Structure-Based Annotation of ADME Gene Variants A Step-by-step Guide on How to Construct quasi-Markov State Models to Study Functional Conformational Changes of Biological Macromolecules Modeling of protein conformational changes with Rosetta guided by limited experimental data TorchDrug: A Powerful and Flexible Machine Learning Platform for Drug Discovery Systematic identification of conditionally folded intrinsically disordered regions by AlphaFold2 A Colab notebook to find protein binding sites on your structure with dMASIF We at LPDI are happy to announce that dMaSIF site is now available on Google Colab thanks to @hamed_khakzad @rneschneuing @befcorreia. It has never been easier to visually identify protein binding sites based on surface features!\nTry it here: https://t.co/xCcEBkkKBd pic.twitter.com/LadZsJ3PFm\n— Casper Goverde (@CasperGoverde) February 21, 2022 TorsionNet: A Deep Neural Network to Rapidly Predict Small-Molecule Torsional Energy Profiles with the Accuracy of Quantum Mechanics Localpdb— a Python package to manage protein structures and their annotations The structural context of PTMs at a proteome wide scale From evolution to folding of repeat proteins Protein-protein interaction prediction for targeted protein degradation SiteMotif: A graph-based algorithm for deriving structural motifs in Protein Ligand binding sites PSnpBind: a database of mutated binding site protein–ligand complexes constructed using a multithreaded virtual screening workflow ","wordCount":"1698","inLanguage":"en","image":"https://folded-weekly.netlify.app/img/14.png","datePublished":"2022-03-03T14:35:29+01:00","dateModified":"2022-03-03T14:35:29+01:00","mainEntityOfPage":{"@type":"WebPage","@id":"https://folded-weekly.netlify.app/posts/14/"},"publisher":{"@type":"Organization","name":"Folded Weekly","logo":{"@type":"ImageObject","url":"https://folded-weekly.netlify.app/favicon.ico"}}}</script></head><body id=top><script>localStorage.getItem("pref-theme")==="dark"?document.body.classList.add("dark"):localStorage.getItem("pref-theme")==="light"?document.body.classList.remove("dark"):window.matchMedia("(prefers-color-scheme: dark)").matches&&document.body.classList.add("dark")</script><header class=header><nav class=nav><div class=logo><a href=https://folded-weekly.netlify.app/ accesskey=h title="Home (Alt + H)"><img src=https://folded-weekly.netlify.app/logo_full.png alt=logo aria-label=logo height=200>Home</a>
<span class=logo-switches><button id=theme-toggle accesskey=t title="(Alt + T)"><svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21 12.79A9 9 0 1111.21 3 7 7 0 0021 12.79z"/></svg><svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><circle cx="12" cy="12" r="5"/><line x1="12" y1="1" x2="12" y2="3"/><line x1="12" y1="21" x2="12" y2="23"/><line x1="4.22" y1="4.22" x2="5.64" y2="5.64"/><line x1="18.36" y1="18.36" x2="19.78" y2="19.78"/><line x1="1" y1="12" x2="3" y2="12"/><line x1="21" y1="12" x2="23" y2="12"/><line x1="4.22" y1="19.78" x2="5.64" y2="18.36"/><line x1="18.36" y1="5.64" x2="19.78" y2="4.22"/></svg></button></span></div><ul id=menu><li><a href=https://folded-weekly.netlify.app/search/ title="Search (Alt + /)" accesskey=/><span>Search</span></a></li><li><a href=https://folded-weekly.netlify.app/archives title=Archive><span>Archive</span></a></li></ul></nav></header><main class=main><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css integrity=sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3+Aro6EYUG4+cU+KJWu/X crossorigin=anonymous><script defer src=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js integrity=sha384-g7c+Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI+sEnkvrMWph2EDg4 crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js integrity=sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC+Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa crossorigin=anonymous onload=renderMathInElement(document.body)></script><article class=post-single><header class=post-header><h1 class=post-title><figure class=entry-cover><img loading=lazy src=https://folded-weekly.netlify.app/img/14.png alt=14></figure>Issue 14</h1><div class=post-meta><span title='2022-03-03 14:35:29 +0100 CET'>March 3, 2022</span>&nbsp;·&nbsp;8 min</div></header><div class=toc><details open><summary accesskey=c title="(Alt + C)"><span class=details>Table of Contents</span></summary><div class=inner><ul><li><a href=#using-deep-learning-to-annotate-the-protein-universehttpsdoiorg101038s41587-021-01179-w aria-label="Using deep learning to annotate the protein universe">Using deep learning to annotate the protein universe</a></li><li><a href=#in-silico-evolution-of-nucleic-acid-binding-proteins-from-a-nonfunctional-scaffoldhttpsdoiorg101038s41589-022-00967-y aria-label="In silico evolution of nucleic acid-binding proteins from a nonfunctional scaffold">In silico evolution of nucleic acid-binding proteins from a nonfunctional scaffold</a></li><li><a href=#pyuul-provides-an-interface-between-biological-structures-and-deep-learning-algorithmshttpsdoiorg101038s41467-022-28327-3 aria-label="PyUUL provides an interface between biological structures and deep learning algorithms">PyUUL provides an interface between biological structures and deep learning algorithms</a></li><li><a href=#systematic-evaluation-of-computational-tools-to-predict-the-effects-of-mutations-on-protein-stability-in-the-absence-of-experimental-structureshttpsdoiorg101093bibbbac025 aria-label="Systematic evaluation of computational tools to predict the effects of mutations on protein stability in the absence of experimental structures">Systematic evaluation of computational tools to predict the effects of mutations on protein stability in the absence of experimental structures</a></li><li><a href=#exploring-protein-protein-interactions-at-the-proteome-levelhttpsdoiorg101016jstr202202004 aria-label="Exploring protein-protein interactions at the proteome level">Exploring protein-protein interactions at the proteome level</a></li><li><a href=#figures-of-the-weeks aria-label="Figure(s) of the Week(s)">Figure(s) of the Week(s)</a></li><li><a href=#many-more aria-label="Many more">Many more</a></li></ul></div></details></div><div class=post-content><p>A double issue covering the last two weeks of updates in structural bioinformatics</p><h2 id=using-deep-learning-to-annotate-the-protein-universehttpsdoiorg101038s41587-021-01179-w><a href=https://doi.org/10.1038/s41587-021-01179-w>Using deep learning to annotate the protein universe</a><a hidden class=anchor aria-hidden=true href=#using-deep-learning-to-annotate-the-protein-universehttpsdoiorg101038s41587-021-01179-w>#</a></h2><p>Each year the wealth of protein-coding sequences uncovered by multiple genomic projects increases exponentially. This makes their annotation for functional features challenging but also more and more important. Metagenomic projects are an especially big source of novel protein sequences and while many for sure represent families completly new to us, a much bigger fraction most likely correspond to highly divergent forms of well known families that just escape detection by HMM-based annotation methods. In this work, the authors trained a network on the domain families in Pfam and demonstrate how deep learning allows the annotation and classification of sequences seemingly without clear homologs, expanding the annotation coverage of the protein universe. The method is useful for the annotation of single domain sequences and allowed for the merging of DUF families with others well known. It does not, however, map annotations into the full-length sequence. The authors circumvent the annotation of multi-domain sequences without known domain boundaries by using a sliding window. The <a href=https://github.com/google-research/google-research/tree/master/using_dl_to_annotate_protein_universe>code</a> is freely available and a <a href=https://colab.research.google.com/github/google-research/google-research/blob/master/using_dl_to_annotate_protein_universe/neural_network/Neural_network_accuracy_on_random_seed_split.ipynb>Colab notebook</a> makes it really convenient to play with your favourite protein. One word of caution however: while powerful and indeed superior in sensitivity to HMMs, the network always provides a result; it always finds the closest annotation in embeded space for a given sequence, even if it is really far away.</p><h2 id=in-silico-evolution-of-nucleic-acid-binding-proteins-from-a-nonfunctional-scaffoldhttpsdoiorg101038s41589-022-00967-y><a href=https://doi.org/10.1038/s41589-022-00967-y>In silico evolution of nucleic acid-binding proteins from a nonfunctional scaffold</a><a hidden class=anchor aria-hidden=true href=#in-silico-evolution-of-nucleic-acid-binding-proteins-from-a-nonfunctional-scaffoldhttpsdoiorg101038s41589-022-00967-y>#</a></h2><p>The authors tackle the fascinating problem of designing proteins with a given function, using simulated directed evolution. To make the task even more challenging they demonstrate their approach, Proseeker, on a completely <em>de novo</em> designed repeat protein with no existing function, designed just to optimize its consecutive alpha-helix folding capabilities. To imbue this protein with nucleotide-binding functionality, the authors create an assessment library of phycochemical descriptors describing short peptides from pentatricopeptide repeat proteins, specifically those covering the RNA-binding residues. These descriptors are clustered and any newly evolved protein is compared to the representative descriptors from each cluster to determine its fitness. Two residues are mutated at each step in the process, with residues around the first mutation site getting a small boost in being selected as the second mutation site, so as to boost creation of mutational islands and epistatic interactions. Then, selection is done computationally using DP-Bind - an SVM trained to predict DNA binding ability. The top scoring 10% variants are used to generate the next round of mutants. Proseeker is experimentally validated by producing five synthetically evolved proteins and running multiple <em>in vitro</em> and <em>in vivo</em> (with a pretty neat new design) assays, showing that three of them actually do have nucleotide-binding activity, an amazing result. Another nice result of the paper is describing all the different variables that are important for getting this kind of approach to work, and systematically varying each one to show its effect - of course this is only possible since the evolution is <em>in silico</em>!</p><h2 id=pyuul-provides-an-interface-between-biological-structures-and-deep-learning-algorithmshttpsdoiorg101038s41467-022-28327-3><a href=https://doi.org/10.1038/s41467-022-28327-3>PyUUL provides an interface between biological structures and deep learning algorithms</a><a hidden class=anchor aria-hidden=true href=#pyuul-provides-an-interface-between-biological-structures-and-deep-learning-algorithmshttpsdoiorg101038s41467-022-28327-3>#</a></h2><p>For the structural biologists who were not born with Machine Learning or fancy Neural Networks (NN), using these new methods can be sometimes intimidating. pyUUL is one of those tools that will help sand down some of the rough edges and make you finally jump on the bandwagon like the other cool kids. The aim of this package is to transform any 3D structures of your choice (protein, small molecules or nucleic acids) into a meaningful representations (voxel-based, surface or volumetric point-cloud) directly exploitable by any NN architecture. At the end, this idea is simple, but nonetheless very powerful. The authors also present multiple test cases, ranging from alpha helix identification, structure-based clustering and small-molecule optimization. Based on PyTorch, all the operations can be run on both CPU and GPU. The package is available <a href=https://bitbucket.org/grogdrinker/pyuul>here</a> and more importantly it comes with <a href=https://pyuul.readthedocs.io>documentation</a> describing some of the examples presented in the papers.</p><h2 id=systematic-evaluation-of-computational-tools-to-predict-the-effects-of-mutations-on-protein-stability-in-the-absence-of-experimental-structureshttpsdoiorg101093bibbbac025><a href=https://doi.org/10.1093/bib/bbac025>Systematic evaluation of computational tools to predict the effects of mutations on protein stability in the absence of experimental structures</a><a hidden class=anchor aria-hidden=true href=#systematic-evaluation-of-computational-tools-to-predict-the-effects-of-mutations-on-protein-stability-in-the-absence-of-experimental-structureshttpsdoiorg101093bibbbac025>#</a></h2><p>Methods for the prediction of protein stability upon mutation are usually evaluated and developed with experimental structures in mind. With the vast majority of structures for proteins being models and predictions, the authors tackled the question of how common mutation stability prediction tools would perform on protein structure models. They evaluated the performance of 14 protein stability predictors on a self-created data set of models based on a curated subset of structures in the ProTherm data set using MODELLER and AlphaFold2. The authors found that the protein stability predictions deteriorate for homology models with a target-template identity below 40%; especially for machine learning and structure based methods. The predictions on AlphaFold2 models were mostly consistent with predictions on experimental structures, which opens up stability predictions with the same accuracy as in experimental structures to all proteins. A limitation we see here is the entanglement of the evaluation of the models of proteins in the ProTherm data base, which is usually also used for the development of stability prediction methods.</p><h2 id=exploring-protein-protein-interactions-at-the-proteome-levelhttpsdoiorg101016jstr202202004><a href=https://doi.org/10.1016/j.str.2022.02.004>Exploring protein-protein interactions at the proteome level</a><a hidden class=anchor aria-hidden=true href=#exploring-protein-protein-interactions-at-the-proteome-levelhttpsdoiorg101016jstr202202004>#</a></h2><p>The biological activity of proteins is defined by how they interact with other biological molecules, from small ligands, to nucleic acids, sugars and other proteins. Usually proteins do not act by themselves, and form strong or transient complexes with other proteins, building large macromolecular machines or transient interaction networks. It is thus essential to not only understand the three-dimensional structure of individual proteins but also in complex with their interaction partners. In this review, Elhabashy <em>et al.</em> discuss the different approaches available to analyse, predict and model such interactions at a large-scale. From experimental methods, to computational and hybrid approaches, they provide a concise but extremely informative and insightful state-of-the-art for the modelling of protein-protein interactions at the proteome level, also highlighting the recent role of machine learning in such tasks.</p><h2 id=figures-of-the-weeks>Figure(s) of the Week(s)<a hidden class=anchor aria-hidden=true href=#figures-of-the-weeks>#</a></h2><figure><img loading=lazy src=https://folded-weekly.netlify.app/img/14_fotw_1.jpg></figure><blockquote><p>Figure 1. Molecular machinery of life
Gallery of supramolecular complexes formed through protein-protein interactions. PDB IDs of shown structures from the left to right are as follows 1S5L: photosynthetic oxygen evolving center; 3J8K: tilted state of actin T2; 2GLS: glutamine synthetase; 3JA6: novel kinase conformational switch in bacterial chemotaxis signaling; 6R3Q: membrane adenylyl cyclase bound to an activated stimulatory G protein; 3JB9: yeast spliceosome; 1FNT: 20s proteasome from yeast in complex with the proteasome activator pa26 from Trypanosoma brucei; 3JBT: the Apaf-1 apoptosome; 7CGO: the flagellar motor-hook complex from Salmonella; 2GJX: human β-hexosaminidase A; 1I6H: RNA polymerase II elongation complex; 4V2T: membrane-embedded pleurotolysin pore with 13-fold symmetry; 5IDE: GluA2/3 AMPA receptor heterotetramer (model I); 1AV1: human apolipoprotein A-I; 4PJ1: human mitochondrial chaperonin symmetrical ‘football’ complex; 5LNK: ovine respiratory complex I; 6VQ6: mammalian V-ATPase from rat brain; 1IGT: immunoglobulin; 1BUW: s-nitroso-nitrosyl human hemoglobin A; 6ASY: human endoplasmic reticulum chaperone BiP; 3J9K: dark apoptosome in complex with the Dronc CARD domain; 4V60: rat liver half-vault; 3J9J: capsaicin receptor TRPV1; and 2GBL: circadian clock protein KaiC</p></blockquote><p><strong>Source: <a href=https://doi.org/10.1016/j.str.2022.02.004>Exploring protein-protein interactions at the proteome level</a></strong></p><hr><figure><img loading=lazy src=https://folded-weekly.netlify.app/img/14_fotw_2.jpg></figure><blockquote><p>Fig. 3. Illustration of the LoopGrafter workflow. The server starts with the input of the PDB files or codes of two structures (orange): (i) the loop receptor (scaffold) and (ii) the loop donator (insert). The tool then calculates the secondary structures of all the loops, assesses the protein flexibility using normal mode analysis, and performs a superimposition of the two structures (blue). Once the user selects the loops of interest, the loops are paired. The suitable loop boundaries are then explored based on geometric restrictions so that maximal sequence diversity is probed in the designed grafted variants. The possible solutions are collected as a list of sequences, and the structures are then modeled and ranked by the respective stability scores obtained from Rosetta and MODELLER (yellow). The web service is provided free of charge for non-commercial use: <a href=https://loschmidt.chemi.muni.cz/loopgrafter/>https://loschmidt.chemi.muni.cz/loopgrafter/</a>.</p></blockquote><p><strong>Source: <a href=https://doi.org/10.1016/j.addr.2022.114143>Tools for computational design and high-throughput screening of therapeutic enzymes</a></strong></p><h2 id=many-more>Many more<a hidden class=anchor aria-hidden=true href=#many-more>#</a></h2><ul><li><a href=https://doi.org/10.1093/nar/gkab1082>DisProt in 2022: improved quality and accessibility of protein intrinsic disorder annotation</a></li><li><a href=https://doi.org/10.1101/2022.01.20.477079>Accurate self-assessment in 2D: protein contact map quality estimation by deep evolutionary reconciliation</a></li><li><a href=https://doi.org/10.3390/ijms23041977>Application of Homology Modeling by Enhanced Profile–Profile Alignment and Flexible-Fitting Simulation to Cryo-EM Based Structure Determination</a></li><li><a href=https://doi.org/10.1093/bioinformatics/btac020>ProteinBERT: a universal deep-learning model of protein sequence and function</a></li><li><a href=https://doi.org/10.1021/acs.jcim.1c01335>MeCOM: A Method for Comparing Three-Dimensional Metalloenzyme Active Sites</a></li><li><a href=https://doi.org/10.1039/D1SC05259D>Model agnostic generation of counterfactual explanations for molecules</a></li><li><a href=https://doi.org/10.1016/j.imu.2022.100880>Molecular modeling in drug discovery</a></li><li><a href=https://doi.org/10.1101/2022.02.11.480122>Ancestral reconstruction of duplicated signaling proteins reveals the evolution of signaling specificity</a></li><li><a href=https://doi.org/10.1016/j.addr.2022.114143>Tools for computational design and high-throughput screening of therapeutic enzymes</a></li><li><a href=https://doi.org/10.1038/s41586-022-04417-6>Visualizing protein breathing motions associated with aromatic ring flipping</a></li><li><a href=https://doi.org/10.1093/jmicro/dfab041>Forty years in cryoEM of membrane proteins</a></li><li>The clinical variant interpretation portal DECIPHER now has AlphaFold structures:</li></ul><blockquote class=twitter-tweet><p lang=en dir=ltr>DECIPHER portal for clinical variant interpretation (free-to-use) now contains AlphaFold structures, annotated with known pathogenic variants, displayed interactively. <a href=https://t.co/rgbBXB5T4M>https://t.co/rgbBXB5T4M</a> <a href=https://t.co/LLKmWs4Clf>pic.twitter.com/LLKmWs4Clf</a></p>&mdash; Matt Hurles (@mehurles) <a href="https://twitter.com/mehurles/status/1494613332896194585?ref_src=twsrc%5Etfw">February 18, 2022</a></blockquote><script async src=https://platform.twitter.com/widgets.js></script><ul><li><a href=https://doi.org/10.1038/s41467-022-28528-w>Structural basis for SARS-CoV-2 Delta variant recognition of ACE2 receptor and broadly neutralizing antibodies</a></li><li><a href=https://doi.org/10.3390/jpm12020263>SWAAT Bioinformatics Workflow for Protein Structure-Based Annotation of ADME Gene Variants</a></li><li><a href=https://doi.org/10.26434/chemrxiv-2022-5gfvq>A Step-by-step Guide on How to Construct quasi-Markov State Models to Study Functional Conformational Changes of Biological Macromolecules</a></li><li><a href=https://doi.org/10.1101/2022.02.14.480383>Modeling of protein conformational changes with Rosetta guided by limited experimental data</a></li><li><a href=https://doi.org/10.48550/arXiv.2202.08320>TorchDrug: A Powerful and Flexible Machine Learning Platform for Drug Discovery</a></li><li><a href=https://doi.org/10.1101/2022.02.18.481080>Systematic identification of conditionally folded intrinsically disordered regions by AlphaFold2</a></li><li>A Colab notebook to find protein binding sites on your structure with dMASIF</li></ul><blockquote class=twitter-tweet><p lang=en dir=ltr>We at LPDI are happy to announce that dMaSIF site is now available on Google Colab thanks to <a href="https://twitter.com/hamed_khakzad?ref_src=twsrc%5Etfw">@hamed_khakzad</a> <a href="https://twitter.com/rneschneuing?ref_src=twsrc%5Etfw">@rneschneuing</a> <a href="https://twitter.com/befcorreia?ref_src=twsrc%5Etfw">@befcorreia</a>. It has never been easier to visually identify protein binding sites based on surface features!<br><br>Try it here: <a href=https://t.co/xCcEBkkKBd>https://t.co/xCcEBkkKBd</a> <a href=https://t.co/LadZsJ3PFm>pic.twitter.com/LadZsJ3PFm</a></p>&mdash; Casper Goverde (@CasperGoverde) <a href="https://twitter.com/CasperGoverde/status/1495704577232019464?ref_src=twsrc%5Etfw">February 21, 2022</a></blockquote><script async src=https://platform.twitter.com/widgets.js></script><ul><li><a href=https://doi.org/10.1021/acs.jcim.1c01346>TorsionNet: A Deep Neural Network to Rapidly Predict Small-Molecule Torsional Energy Profiles with the Accuracy of Quantum Mechanics</a></li><li><a href=https://doi.org/10.1093/bioinformatics/btac121>Localpdb— a Python package to manage protein structures and their annotations</a></li><li><a href=https://doi.org/10.1101/2022.02.23.481596>The structural context of PTMs at a proteome wide scale</a></li><li><a href=https://doi.org/10.48550/arXiv.2202.12223>From evolution to folding of repeat proteins</a></li><li><a href=https://doi.org/10.1101/2022.02.25.481776>Protein-protein interaction prediction for targeted protein degradation</a></li><li><a href=https://doi.org/10.1371/journal.pcbi.1009901>SiteMotif: A graph-based algorithm for deriving structural motifs in Protein Ligand binding sites</a></li><li><a href=https://doi.org/10.1186/s13321-021-00573-5>PSnpBind: a database of mutated binding site protein–ligand complexes constructed using a multithreaded virtual screening workflow</a></li></ul></div><div class=subscribe><form action=https://buttondown.email/api/emails/embed-subscribe/FoldedWeekly method=post target=popupwindow onsubmit='window.open("https://buttondown.email/FoldedWeekly","popupwindow")' class=embeddable-buttondown-form><input type=email name=email id=bd-email placeholder="Enter your email">
<input type=submit value=Subscribe><p><a href=https://buttondown.email target=_blank id=bd-link>Powered by Buttondown</a></p></form></div><footer class=post-footer><nav class=paginav><a class=prev href=https://folded-weekly.netlify.app/posts/15/><span class=title>« Prev Page</span><br><span>Issue 15</span></a>
<a class=next href=https://folded-weekly.netlify.app/posts/13/><span class=title>Next Page »</span><br><span>Issue 13</span></a></nav><div class=share-buttons><a target=_blank rel="noopener noreferrer" aria-label="share Issue 14 on twitter" href="https://twitter.com/intent/tweet/?text=Issue%2014&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f14%2f&amp;hashtags="><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM195.519 424.544c135.939.0 210.268-112.643 210.268-210.268.0-3.218.0-6.437-.153-9.502 14.406-10.421 26.973-23.448 36.935-38.314-13.18 5.824-27.433 9.809-42.452 11.648 15.326-9.196 26.973-23.602 32.49-40.92-14.252 8.429-30.038 14.56-46.896 17.931-13.487-14.406-32.644-23.295-53.946-23.295-40.767.0-73.87 33.104-73.87 73.87.0 5.824.613 11.494 1.992 16.858-61.456-3.065-115.862-32.49-152.337-77.241-6.284 10.881-9.962 23.601-9.962 37.088.0 25.594 13.027 48.276 32.95 61.456-12.107-.307-23.448-3.678-33.41-9.196v.92c0 35.862 25.441 65.594 59.311 72.49-6.13 1.686-12.72 2.606-19.464 2.606-4.751.0-9.348-.46-13.946-1.38 9.349 29.426 36.628 50.728 68.965 51.341-25.287 19.771-57.164 31.571-91.8 31.571-5.977.0-11.801-.306-17.625-1.073 32.337 21.15 71.264 33.41 112.95 33.41z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 14 on linkedin" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f14%2f&amp;title=Issue%2014&amp;summary=Issue%2014&amp;source=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f14%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM160.461 423.278V197.561h-75.04v225.717h75.04zm270.539.0V293.839c0-69.333-37.018-101.586-86.381-101.586-39.804.0-57.634 21.891-67.617 37.266v-31.958h-75.021c.995 21.181.0 225.717.0 225.717h75.02V297.222c0-6.748.486-13.492 2.474-18.315 5.414-13.475 17.767-27.434 38.494-27.434 27.135.0 38.007 20.707 38.007 51.037v120.768H431zM123.448 88.722C97.774 88.722 81 105.601 81 127.724c0 21.658 16.264 39.002 41.455 39.002h.484c26.165.0 42.452-17.344 42.452-39.002-.485-22.092-16.241-38.954-41.943-39.002z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 14 on reddit" href="https://reddit.com/submit?url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f14%2f&title=Issue%2014"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM446 265.638c0-22.964-18.616-41.58-41.58-41.58-11.211.0-21.361 4.457-28.841 11.666-28.424-20.508-67.586-33.757-111.204-35.278l18.941-89.121 61.884 13.157c.756 15.734 13.642 28.29 29.56 28.29 16.407.0 29.706-13.299 29.706-29.701.0-16.403-13.299-29.702-29.706-29.702-11.666.0-21.657 6.792-26.515 16.578l-69.105-14.69c-1.922-.418-3.939-.042-5.585 1.036-1.658 1.073-2.811 2.761-3.224 4.686l-21.152 99.438c-44.258 1.228-84.046 14.494-112.837 35.232-7.468-7.164-17.589-11.591-28.757-11.591-22.965.0-41.585 18.616-41.585 41.58.0 16.896 10.095 31.41 24.568 37.918-.639 4.135-.99 8.328-.99 12.576.0 63.977 74.469 115.836 166.33 115.836s166.334-51.859 166.334-115.836c0-4.218-.347-8.387-.977-12.493 14.564-6.47 24.735-21.034 24.735-38.001zM326.526 373.831c-20.27 20.241-59.115 21.816-70.534 21.816-11.428.0-50.277-1.575-70.522-21.82-3.007-3.008-3.007-7.882.0-10.889 3.003-2.999 7.882-3.003 10.885.0 12.777 12.781 40.11 17.317 59.637 17.317 19.522.0 46.86-4.536 59.657-17.321 3.016-2.999 7.886-2.995 10.885.008 3.008 3.011 3.003 7.882-.008 10.889zm-5.23-48.781c-16.373.0-29.701-13.324-29.701-29.698.0-16.381 13.328-29.714 29.701-29.714 16.378.0 29.706 13.333 29.706 29.714.0 16.374-13.328 29.698-29.706 29.698zM160.91 295.348c0-16.381 13.328-29.71 29.714-29.71 16.369.0 29.689 13.329 29.689 29.71.0 16.373-13.32 29.693-29.689 29.693-16.386.0-29.714-13.32-29.714-29.693z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 14 on facebook" href="https://facebook.com/sharer/sharer.php?u=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f14%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H342.978V319.085h66.6l12.672-82.621h-79.272v-53.617c0-22.603 11.073-44.636 46.58-44.636H425.6v-70.34s-32.71-5.582-63.982-5.582c-65.288.0-107.96 39.569-107.96 111.204v62.971h-72.573v82.621h72.573V512h-191.104c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 14 on whatsapp" href="https://api.whatsapp.com/send?text=Issue%2014%20-%20https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f14%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zm-58.673 127.703c-33.842-33.881-78.847-52.548-126.798-52.568-98.799.0-179.21 80.405-179.249 179.234-.013 31.593 8.241 62.428 23.927 89.612l-25.429 92.884 95.021-24.925c26.181 14.28 55.659 21.807 85.658 21.816h.074c98.789.0 179.206-80.413 179.247-179.243.018-47.895-18.61-92.93-52.451-126.81zM263.976 403.485h-.06c-26.734-.01-52.954-7.193-75.828-20.767l-5.441-3.229-56.386 14.792 15.05-54.977-3.542-5.637c-14.913-23.72-22.791-51.136-22.779-79.287.033-82.142 66.867-148.971 149.046-148.971 39.793.014 77.199 15.531 105.329 43.692 28.128 28.16 43.609 65.592 43.594 105.4-.034 82.149-66.866 148.983-148.983 148.984zm81.721-111.581c-4.479-2.242-26.499-13.075-30.604-14.571-4.105-1.495-7.091-2.241-10.077 2.241-2.986 4.483-11.569 14.572-14.182 17.562-2.612 2.988-5.225 3.364-9.703 1.12-4.479-2.241-18.91-6.97-36.017-22.23C231.8 264.15 222.81 249.484 220.198 245s-.279-6.908 1.963-9.14c2.016-2.007 4.48-5.232 6.719-7.847 2.24-2.615 2.986-4.484 4.479-7.472 1.493-2.99.747-5.604-.374-7.846-1.119-2.241-10.077-24.288-13.809-33.256-3.635-8.733-7.327-7.55-10.077-7.688-2.609-.13-5.598-.158-8.583-.158-2.986.0-7.839 1.121-11.944 5.604-4.105 4.484-15.675 15.32-15.675 37.364.0 22.046 16.048 43.342 18.287 46.332 2.24 2.99 31.582 48.227 76.511 67.627 10.685 4.615 19.028 7.371 25.533 9.434 10.728 3.41 20.492 2.929 28.209 1.775 8.605-1.285 26.499-10.833 30.231-21.295 3.732-10.464 3.732-19.431 2.612-21.298-1.119-1.869-4.105-2.99-8.583-5.232z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 14 on telegram" href="https://telegram.me/share/url?text=Issue%2014&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f14%2f"><svg viewBox="2 2 28 28"><path d="M26.49 29.86H5.5a3.37 3.37.0 01-2.47-1 3.35 3.35.0 01-1-2.47V5.48A3.36 3.36.0 013 3 3.37 3.37.0 015.5 2h21A3.38 3.38.0 0129 3a3.36 3.36.0 011 2.46V26.37a3.35 3.35.0 01-1 2.47 3.38 3.38.0 01-2.51 1.02zm-5.38-6.71a.79.79.0 00.85-.66L24.73 9.24a.55.55.0 00-.18-.46.62.62.0 00-.41-.17q-.08.0-16.53 6.11a.59.59.0 00-.41.59.57.57.0 00.43.52l4 1.24 1.61 4.83a.62.62.0 00.63.43.56.56.0 00.4-.17L16.54 20l4.09 3A.9.9.0 0021.11 23.15zM13.8 20.71l-1.21-4q8.72-5.55 8.78-5.55c.15.0.23.0.23.16a.18.18.0 010 .06s-2.51 2.3-7.52 6.8z"/></svg></a></div></footer></article></main><footer class=footer><span>&copy; 2023 <a href=https://folded-weekly.netlify.app/>Folded Weekly</a></span>
<span>Powered by
<a href=https://gohugo.io/ rel="noopener noreferrer" target=_blank>Hugo</a> &
        <a href=https://git.io/hugopapermod rel=noopener target=_blank>PaperMod</a></span></footer><a href=#top aria-label="go to top" title="Go to Top (Alt + G)" class=top-link id=top-link accesskey=g><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentcolor"><path d="M12 6H0l6-6z"/></svg></a><script>let menu=document.getElementById("menu");menu&&(menu.scrollLeft=localStorage.getItem("menu-scroll-position"),menu.onscroll=function(){localStorage.setItem("menu-scroll-position",menu.scrollLeft)}),document.querySelectorAll('a[href^="#"]').forEach(e=>{e.addEventListener("click",function(e){e.preventDefault();var t=this.getAttribute("href").substr(1);window.matchMedia("(prefers-reduced-motion: reduce)").matches?document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView():document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView({behavior:"smooth"}),t==="top"?history.replaceState(null,null," "):history.pushState(null,null,`#${t}`)})})</script><script>var mybutton=document.getElementById("top-link");window.onscroll=function(){document.body.scrollTop>800||document.documentElement.scrollTop>800?(mybutton.style.visibility="visible",mybutton.style.opacity="1"):(mybutton.style.visibility="hidden",mybutton.style.opacity="0")}</script><script>document.getElementById("theme-toggle").addEventListener("click",()=>{document.body.className.includes("dark")?(document.body.classList.remove("dark"),localStorage.setItem("pref-theme","light")):(document.body.classList.add("dark"),localStorage.setItem("pref-theme","dark"))})</script></body></html>