<!doctype html><html lang=en dir=auto><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=robots content="index, follow"><title>Issue 12 | Folded Weekly</title><meta name=keywords content><meta name=description content="This week we cover protein evolution as seen by language models, molecular design by reinforcement learning, and molecular electronics to measure binding"><meta name=author content><link rel=canonical href=https://folded-weekly.netlify.app/posts/12/><link crossorigin=anonymous href=/assets/css/stylesheet.min.0caf3925a7f1eae074a9aadc5d80291998cf748272cb5b0e65ca8199d38d7909.css integrity="sha256-DK85Jafx6uB0qarcXYApGZjPdIJyy1sOZcqBmdONeQk=" rel="preload stylesheet" as=style><link rel=preload href=logo_full.png as=image><script defer crossorigin=anonymous src=/assets/js/highlight.min.b95bacdc39e37a332a9f883b1e78be4abc1fdca2bc1f2641f55e3cd3dabd4d61.js integrity="sha256-uVus3DnjejMqn4g7Hni+Srwf3KK8HyZB9V4809q9TWE=" onload=hljs.initHighlightingOnLoad()></script>
<link rel=icon href=https://folded-weekly.netlify.app/favicon.ico><link rel=icon type=image/png sizes=16x16 href=https://folded-weekly.netlify.app/favicon-16x16.png><link rel=icon type=image/png sizes=32x32 href=https://folded-weekly.netlify.app/favicon-32x32.png><link rel=apple-touch-icon href=https://folded-weekly.netlify.app/apple-touch-icon.png><link rel=mask-icon href=https://folded-weekly.netlify.app/favicon-16x16.png><meta name=theme-color content="#2e2e33"><meta name=msapplication-TileColor content="#2e2e33"><link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;0,700;1,400&&family=Noto+Serif&family=Playfair+Display:wght@500&display=swap" rel=stylesheet><meta name=generator content="Hugo 0.111.3"><noscript><style>#theme-toggle,.top-link{display:none}</style><style>@media(prefers-color-scheme:dark){:root{--theme:rgb(29, 30, 32);--entry:rgb(46, 46, 51);--primary:rgb(218, 218, 219);--secondary:rgb(155, 156, 157);--tertiary:rgb(65, 66, 68);--content:rgb(196, 196, 197);--hljs-bg:rgb(46, 46, 51);--code-bg:rgb(55, 56, 62);--border:rgb(51, 51, 51)}.list{background:var(--theme)}.list:not(.dark)::-webkit-scrollbar-track{background:0 0}.list:not(.dark)::-webkit-scrollbar-thumb{border-color:var(--theme)}}</style></noscript><meta property="og:title" content="Issue 12"><meta property="og:description" content="This week we cover protein evolution as seen by language models, molecular design by reinforcement learning, and molecular electronics to measure binding"><meta property="og:type" content="article"><meta property="og:url" content="https://folded-weekly.netlify.app/posts/12/"><meta property="og:image" content="https://folded-weekly.netlify.app/img/12.png"><meta property="article:section" content="posts"><meta property="article:published_time" content="2022-02-09T08:50:49+01:00"><meta property="article:modified_time" content="2022-02-09T08:50:49+01:00"><meta property="og:site_name" content="Folded Weekly"><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://folded-weekly.netlify.app/img/12.png"><meta name=twitter:title content="Issue 12"><meta name=twitter:description content="This week we cover protein evolution as seen by language models, molecular design by reinforcement learning, and molecular electronics to measure binding"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Posts","item":"https://folded-weekly.netlify.app/posts/"},{"@type":"ListItem","position":2,"name":"Issue 12","item":"https://folded-weekly.netlify.app/posts/12/"}]}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","headline":"Issue 12","name":"Issue 12","description":"This week we cover protein evolution as seen by language models, molecular design by reinforcement learning, and molecular electronics to measure binding","keywords":[],"articleBody":"Evolutionary velocity with protein language models predicts evolutionary dynamics of diverse proteins Protein language models are revolutionizing protein bioinformatics. From the prediction of protein sequence features and the classification of protein families, to important contributions to protein structure prediction. In this paper, Hie et al go a step further and use them to model protein sequence evolution as a ‚Äúwalk through the protein sequence landscape‚Äù. Based on the ESM-1b protein language model, which the authors say learnt general protein evolutionary patterns, they developed an approach that used distances in embedded space to build a sequence similarity network on top of which they map a ‚Äúvector field‚Äù that translates into the evolutionary dynamics within the sequences analysed. The approach starts by the computation of the sequence embeddings for a target set of sequences using the language model. KNN as implemented in UMAP is then used to find the neighbours of each sequence in embedded space, mimicking a sequence similarity network. Then, the change in the language model pseudo-likelihood in each edge is used to assign a ‚Äúvelocity‚Äù vector whose direction represents an evolutionary transition. While it is not clear to me why this last step has the evolutionary meaning claimed, the authors demonstrate that for different protein families the internal structure of resulting network and the vector field generated captures their known evolutionary patterns. What makes this work even more exciting is that the method is made available as a python package (evolocity), which relies on Scanpy and anndata, two other packages that facilitate the analysis of large multidimensional annotated data. The package comes with multiple tutorials that can be run in Google Colab; this makes it easier to explore it for our favourite sets of sequences. But be warned: you may run into crashes due to memory overload in Colab. Nevertheless, it is exciting that such a tool is now made easily available and, while there is some amount of flexibility within the package, the expansion to other clustering methods and further language models would make it even more rich.\nScalable Fragment-Based 3D Molecular Design with Reinforcement Learning The authors describe an interesting molecule design method that combines Markov decision processes with machine learning. It consists in a hierarchical agent that sequentially places molecular fragments with predefined shapes in 3D space. They compare this methodology with the intuition of an organic chemist who, when faced with a task to modify a molecule, would think of it in terms of adding new fragments and functional groups rather than single atoms. This method allows building molecules with more than 100 atoms, a task that other approaches that take the 3D structure into consideration usually fail to accomplish. Moreover, they suggest that it is possible to optimize different properties during the design process, like hydrophilicity or the affinity with a specific pocket. This makes this method extremely versatile, leading to many potential applications such as: small molecule and peptide drug design or design of ligands and organocatalysts.\nMolecular electronics sensors on a scalable semiconductor chip: A platform for single-molecule measurement of binding kinetics and enzyme activity The future is now! The authors realise the 70s‚Äô dream of using single molecules as circuit elements, with a CMOS-integrated circuit chip consisting of an array of nanoelectrode-based single-molecule sensors ‚Äúbridged‚Äù by a molecular wire between the nanoelectrodes. The biomolecule of interest is conjugated to the bridge molecule to act as a probe (a process that takes 10 seconds), the target molecule is added into the equation, and then the current through the bridge is monitored - if the target molecule interacts with the probe then this can be seen via the current pulses. And not just seen either - Hidden Markov models are used to analyse the current signals, with bound and unbound being the hidden states, and by comparing the fraction of bound vs unbound across different target concentrations they obtain pretty accurate estimates for the binding affinity $K_d$ as well! The authors demonstrate an impressive variety of applications - DNA hybridization of complementary oligos (including determining the melting temperature of the DNA duplex, and showing sensitivity to introduced mismatches), protein and small-molecule binding, DNA aptamer-target binding (shown using aptamers designed to bind SARS-CoV2 proteins), and enzyme activity (shown using a CRISPR/Cas system with the guide RNA targeting the Spike protein gene, and by distinguishing between DNA polymerase enzyme incorporation events) among others. They even show that adding a bunch of saliva or other highly complex contaminants doesn‚Äôt mess up sensing, meaning it‚Äôs perfect for diagnostic tests. Overall, the chip is sensitive and specific, can be multiplexed to target many kinds of interactions and molecules, highly scalable way past Moore‚Äôs law, fast (seconds) and cheap (~1$). Looks like a bright future for obtaining high quality drug-target binding affinity data to train our methods!\nFigure of the Week Source: Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning\nSome more Cryo-EM and artificial intelligence visualize endogenous protein community members What stabilizes pre-folded structures in the intrinsically disordered Œ±-helical binding motifs? Riemannian Geometry and Molecular Surfaces I: Spectrum of the Laplacian Computational Modeling of Molecular Structures Guided by Hydrogen-Exchange Data Current structure predictors are not learning the physics of protein folding AlphaDesign: A graph protein design method and benchmark on AlphaFoldDB DisEnrich: Database of Enriched Regions in Human Dark Proteome EViS: An Enhanced Virtual Screening Approach Based on Pocket‚ÄìLigand Similarity Assessment of AlphaFold2 residue conformations for human proteins ChemPlot, a Python library for chemical space visualization SPOT-Contact-LM: Improving Single-Sequence-Based Prediction of Protein Contact Map using a Transformer Language Model A Springer book on Artificial Intelligence in Drug Design UniProt has a Machine Learning Challenge aimed at creating computational methods to predict metal binding sites across the whole of UniProtKB Some resources on the latest hot topic, protein design via hallucination: A video tutorial and ColabFold support Adding support for binder hallucination if anyone wants to try! (Code is very experimental, not intended for practical use... only use for art/science) üòÄhttps://t.co/OOPp8kSu2Z pic.twitter.com/WUu9LGKIwp\n‚Äî Sergey Ovchinnikov üá∫üá¶ (@sokrypton) February 3, 2022 ","wordCount":"1001","inLanguage":"en","image":"https://folded-weekly.netlify.app/img/12.png","datePublished":"2022-02-09T08:50:49+01:00","dateModified":"2022-02-09T08:50:49+01:00","mainEntityOfPage":{"@type":"WebPage","@id":"https://folded-weekly.netlify.app/posts/12/"},"publisher":{"@type":"Organization","name":"Folded Weekly","logo":{"@type":"ImageObject","url":"https://folded-weekly.netlify.app/favicon.ico"}}}</script></head><body id=top><script>localStorage.getItem("pref-theme")==="dark"?document.body.classList.add("dark"):localStorage.getItem("pref-theme")==="light"?document.body.classList.remove("dark"):window.matchMedia("(prefers-color-scheme: dark)").matches&&document.body.classList.add("dark")</script><header class=header><nav class=nav><div class=logo><a href=https://folded-weekly.netlify.app/ accesskey=h title="Home (Alt + H)"><img src=https://folded-weekly.netlify.app/logo_full.png alt=logo aria-label=logo height=200>Home</a>
<span class=logo-switches><button id=theme-toggle accesskey=t title="(Alt + T)"><svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21 12.79A9 9 0 1111.21 3 7 7 0 0021 12.79z"/></svg><svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><circle cx="12" cy="12" r="5"/><line x1="12" y1="1" x2="12" y2="3"/><line x1="12" y1="21" x2="12" y2="23"/><line x1="4.22" y1="4.22" x2="5.64" y2="5.64"/><line x1="18.36" y1="18.36" x2="19.78" y2="19.78"/><line x1="1" y1="12" x2="3" y2="12"/><line x1="21" y1="12" x2="23" y2="12"/><line x1="4.22" y1="19.78" x2="5.64" y2="18.36"/><line x1="18.36" y1="5.64" x2="19.78" y2="4.22"/></svg></button></span></div><ul id=menu><li><a href=https://folded-weekly.netlify.app/search/ title="Search (Alt + /)" accesskey=/><span>Search</span></a></li><li><a href=https://folded-weekly.netlify.app/archives title=Archive><span>Archive</span></a></li></ul></nav></header><main class=main><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css integrity=sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3+Aro6EYUG4+cU+KJWu/X crossorigin=anonymous><script defer src=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js integrity=sha384-g7c+Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI+sEnkvrMWph2EDg4 crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js integrity=sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC+Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa crossorigin=anonymous onload=renderMathInElement(document.body)></script><article class=post-single><header class=post-header><h1 class=post-title><figure class=entry-cover><img loading=lazy src=https://folded-weekly.netlify.app/img/12.png alt=12></figure>Issue 12</h1><div class=post-meta><span title='2022-02-09 08:50:49 +0100 CET'>February 9, 2022</span>&nbsp;¬∑&nbsp;5 min</div></header><div class=toc><details open><summary accesskey=c title="(Alt + C)"><span class=details>Table of Contents</span></summary><div class=inner><ul><li><a href=#evolutionary-velocity-with-protein-language-models-predicts-evolutionary-dynamics-of-diverse-proteinshttpsdoiorg101016jcels202201003 aria-label="Evolutionary velocity with protein language models predicts evolutionary dynamics of diverse proteins">Evolutionary velocity with protein language models predicts evolutionary dynamics of diverse proteins</a></li><li><a href=#scalable-fragment-based-3d-molecular-design-with-reinforcement-learninghttpsarxivorgabs220200658 aria-label="Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning">Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning</a></li><li><a href=#molecular-electronics-sensors-on-a-scalable-semiconductor-chip-a-platform-for-single-molecule-measurement-of-binding-kinetics-and-enzyme-activityhttpsdoiorg101073pnas2112812119 aria-label="Molecular electronics sensors on a scalable semiconductor chip: A platform for single-molecule measurement of binding kinetics and enzyme activity">Molecular electronics sensors on a scalable semiconductor chip: A platform for single-molecule measurement of binding kinetics and enzyme activity</a></li><li><a href=#figure-of-the-week aria-label="Figure of the Week">Figure of the Week</a></li><li><a href=#some-more aria-label="Some more">Some more</a></li></ul></div></details></div><div class=post-content><h2 id=evolutionary-velocity-with-protein-language-models-predicts-evolutionary-dynamics-of-diverse-proteinshttpsdoiorg101016jcels202201003><a href=https://doi.org/10.1016/j.cels.2022.01.003>Evolutionary velocity with protein language models predicts evolutionary dynamics of diverse proteins</a><a hidden class=anchor aria-hidden=true href=#evolutionary-velocity-with-protein-language-models-predicts-evolutionary-dynamics-of-diverse-proteinshttpsdoiorg101016jcels202201003>#</a></h2><p>Protein language models are revolutionizing protein bioinformatics. From the prediction of protein sequence features and the classification of protein families, to important contributions to protein structure prediction. In this paper, Hie <em>et al</em> go a step further and use them to model protein sequence evolution as a &ldquo;walk through the protein sequence landscape&rdquo;. Based on the ESM-1b protein language model, which the authors say learnt general protein evolutionary patterns, they developed an approach that used distances in embedded space to build a sequence similarity network on top of which they map a &ldquo;vector field&rdquo; that translates into the evolutionary dynamics within the sequences analysed. The approach starts by the computation of the sequence embeddings for a target set of sequences using the language model. KNN as implemented in UMAP is then used to find the neighbours of each sequence in embedded space, mimicking a sequence similarity network. Then, the change in the language model pseudo-likelihood in each edge is used to assign a &ldquo;velocity&rdquo; vector whose direction represents an evolutionary transition. While it is not clear to me why this last step has the evolutionary meaning claimed, the authors demonstrate that for different protein families the internal structure of resulting network and the vector field generated captures their known evolutionary patterns. What makes this work even more exciting is that the method is made available as a python package (<a href=https://github.com/brianhie/evolocity>evolocity</a>), which relies on <a href=https://github.com/theislab/scanpy/>Scanpy</a> and <a href=https://github.com/theislab/anndata>anndata</a>, two other packages that facilitate the analysis of large multidimensional annotated data. The package comes with multiple tutorials that can be run in Google Colab; this makes it easier to explore it for our favourite sets of sequences. But be warned: you may run into crashes due to memory overload in Colab. Nevertheless, it is exciting that such a tool is now made easily available and, while there is some amount of flexibility within the package, the expansion to other clustering methods and further language models would make it even more rich.</p><h2 id=scalable-fragment-based-3d-molecular-design-with-reinforcement-learninghttpsarxivorgabs220200658><a href=https://arxiv.org/abs/2202.00658>Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning</a><a hidden class=anchor aria-hidden=true href=#scalable-fragment-based-3d-molecular-design-with-reinforcement-learninghttpsarxivorgabs220200658>#</a></h2><p>The authors describe an interesting molecule design method that combines Markov decision processes with machine learning. It consists in a hierarchical agent that sequentially places molecular fragments with predefined shapes in 3D space. They compare this methodology with the intuition of an organic chemist who, when faced with a task to modify a molecule, would think of it in terms of adding new fragments and functional groups rather than single atoms. This method allows building molecules with more than 100 atoms, a task that other approaches that take the 3D structure into consideration usually fail to accomplish. Moreover, they suggest that it is possible to optimize different properties during the design process, like hydrophilicity or the affinity with a specific pocket. This makes this method extremely versatile, leading to many potential applications such as: small molecule and peptide drug design or design of ligands and organocatalysts.</p><h2 id=molecular-electronics-sensors-on-a-scalable-semiconductor-chip-a-platform-for-single-molecule-measurement-of-binding-kinetics-and-enzyme-activityhttpsdoiorg101073pnas2112812119><a href=https://doi.org/10.1073/pnas.2112812119>Molecular electronics sensors on a scalable semiconductor chip: A platform for single-molecule measurement of binding kinetics and enzyme activity</a><a hidden class=anchor aria-hidden=true href=#molecular-electronics-sensors-on-a-scalable-semiconductor-chip-a-platform-for-single-molecule-measurement-of-binding-kinetics-and-enzyme-activityhttpsdoiorg101073pnas2112812119>#</a></h2><p>The future is now! The authors realise the 70s&rsquo; dream of using single molecules as circuit elements, with a CMOS-integrated circuit chip consisting of an array of nanoelectrode-based single-molecule sensors &ldquo;bridged&rdquo; by a molecular wire between the nanoelectrodes. The biomolecule of interest is conjugated to the bridge molecule to act as a probe (a process that takes 10 seconds), the target molecule is added into the equation, and then the current through the bridge is monitored - if the target molecule interacts with the probe then this can be seen via the current pulses. And not just seen either - Hidden Markov models are used to analyse the current signals, with bound and unbound being the hidden states, and by comparing the fraction of bound vs unbound across different target concentrations they obtain pretty accurate estimates for the binding affinity $K_d$ as well! The authors demonstrate an impressive variety of applications - DNA hybridization of complementary oligos (including determining the melting temperature of the DNA duplex, and showing sensitivity to introduced mismatches), protein and small-molecule binding, DNA aptamer-target binding (shown using aptamers designed to bind SARS-CoV2 proteins), and enzyme activity (shown using a CRISPR/Cas system with the guide RNA targeting the Spike protein gene, and by distinguishing between DNA polymerase enzyme incorporation events) among others. They even show that adding a bunch of saliva or other highly complex contaminants doesn&rsquo;t mess up sensing, meaning it&rsquo;s perfect for diagnostic tests. Overall, the chip is sensitive and specific, can be multiplexed to target many kinds of interactions and molecules, highly scalable way past Moore&rsquo;s law, fast (seconds) and cheap (~1$). Looks like a bright future for obtaining high quality drug-target binding affinity data to train our methods!</p><h2 id=figure-of-the-week>Figure of the Week<a hidden class=anchor aria-hidden=true href=#figure-of-the-week>#</a></h2><figure><img loading=lazy src=https://folded-weekly.netlify.app/img/12_fotw.png></figure><p><strong>Source: <a href=https://arxiv.org/abs/2202.00658>Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning</a></strong></p><h2 id=some-more>Some more<a hidden class=anchor aria-hidden=true href=#some-more>#</a></h2><ul><li><a href=https://doi.org/10.1016/j.str.2022.01.001>Cryo-EM and artificial intelligence visualize endogenous protein community members</a></li><li><a href=https://doi.org/10.1101/2022.01.28.478151>What stabilizes pre-folded structures in the intrinsically disordered Œ±-helical binding motifs?</a></li><li><a href=https://arxiv.org/abs/2201.04230>Riemannian Geometry and Molecular Surfaces I: Spectrum of the Laplacian</a></li><li><a href=https://doi.org/10.1021/jasms.1c00328>Computational Modeling of Molecular Structures Guided by Hydrogen-Exchange Data</a></li><li><a href=https://doi.org/10.1093/bioinformatics/btab881>Current structure predictors are not learning the physics of protein folding</a></li><li><a href=https://arxiv.org/abs/2202.01079>AlphaDesign: A graph protein design method and benchmark on AlphaFoldDB</a></li><li><a href=https://doi.org/10.1093/bioinformatics/btac051>DisEnrich: Database of Enriched Regions in Human Dark Proteome</a></li><li><a href=https://doi.org/10.1021/acs.jcim.1c00944>EViS: An Enhanced Virtual Screening Approach Based on Pocket‚ÄìLigand Similarity</a></li><li><a href=https://www.biorxiv.org/content/10.1101/2022.01.28.478137v1>Assessment of AlphaFold2 residue conformations for human proteins</a></li><li><a href=https://doi.org/10.26434/chemrxiv-2021-3zv3k-v2>ChemPlot, a Python library for chemical space visualization</a></li><li><a href=https://doi.org/10.1093/bioinformatics/btac053>SPOT-Contact-LM: Improving Single-Sequence-Based Prediction of Protein Contact Map using a Transformer Language Model</a></li><li>A Springer book on <a href=https://link.springer.com/book/10.1007/978-1-0716-1787-8>Artificial Intelligence in Drug Design</a></li><li>UniProt has a <a href=http://insideuniprot.blogspot.com/2022/02/the-uniprot-metal-binding-site-machine.html>Machine Learning Challenge</a> aimed at creating computational methods to predict metal binding sites across the whole of UniProtKB</li><li>Some resources on the latest hot topic, protein design via hallucination: <a href="https://www.youtube.com/watch?v=2HmXwlKWMVs">A video tutorial</a> and ColabFold support</li></ul><blockquote class=twitter-tweet><p lang=en dir=ltr>Adding support for binder hallucination if anyone wants to try! (Code is very experimental, not intended for practical use... only use for art/science) üòÄ<a href=https://t.co/OOPp8kSu2Z>https://t.co/OOPp8kSu2Z</a> <a href=https://t.co/WUu9LGKIwp>pic.twitter.com/WUu9LGKIwp</a></p>&mdash; Sergey Ovchinnikov üá∫üá¶ (@sokrypton) <a href="https://twitter.com/sokrypton/status/1489279054649991172?ref_src=twsrc%5Etfw">February 3, 2022</a></blockquote><script async src=https://platform.twitter.com/widgets.js></script></div><div class=subscribe><form action=https://buttondown.email/api/emails/embed-subscribe/FoldedWeekly method=post target=popupwindow onsubmit='window.open("https://buttondown.email/FoldedWeekly","popupwindow")' class=embeddable-buttondown-form><input type=email name=email id=bd-email placeholder="Enter your email">
<input type=submit value=Subscribe><p><a href=https://buttondown.email target=_blank id=bd-link>Powered by Buttondown</a></p></form></div><footer class=post-footer><nav class=paginav><a class=prev href=https://folded-weekly.netlify.app/posts/13/><span class=title>¬´ Prev Page</span><br><span>Issue 13</span></a>
<a class=next href=https://folded-weekly.netlify.app/posts/11/><span class=title>Next Page ¬ª</span><br><span>Issue 11</span></a></nav><div class=share-buttons><a target=_blank rel="noopener noreferrer" aria-label="share Issue 12 on twitter" href="https://twitter.com/intent/tweet/?text=Issue%2012&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f12%2f&amp;hashtags="><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM195.519 424.544c135.939.0 210.268-112.643 210.268-210.268.0-3.218.0-6.437-.153-9.502 14.406-10.421 26.973-23.448 36.935-38.314-13.18 5.824-27.433 9.809-42.452 11.648 15.326-9.196 26.973-23.602 32.49-40.92-14.252 8.429-30.038 14.56-46.896 17.931-13.487-14.406-32.644-23.295-53.946-23.295-40.767.0-73.87 33.104-73.87 73.87.0 5.824.613 11.494 1.992 16.858-61.456-3.065-115.862-32.49-152.337-77.241-6.284 10.881-9.962 23.601-9.962 37.088.0 25.594 13.027 48.276 32.95 61.456-12.107-.307-23.448-3.678-33.41-9.196v.92c0 35.862 25.441 65.594 59.311 72.49-6.13 1.686-12.72 2.606-19.464 2.606-4.751.0-9.348-.46-13.946-1.38 9.349 29.426 36.628 50.728 68.965 51.341-25.287 19.771-57.164 31.571-91.8 31.571-5.977.0-11.801-.306-17.625-1.073 32.337 21.15 71.264 33.41 112.95 33.41z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 12 on linkedin" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f12%2f&amp;title=Issue%2012&amp;summary=Issue%2012&amp;source=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f12%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM160.461 423.278V197.561h-75.04v225.717h75.04zm270.539.0V293.839c0-69.333-37.018-101.586-86.381-101.586-39.804.0-57.634 21.891-67.617 37.266v-31.958h-75.021c.995 21.181.0 225.717.0 225.717h75.02V297.222c0-6.748.486-13.492 2.474-18.315 5.414-13.475 17.767-27.434 38.494-27.434 27.135.0 38.007 20.707 38.007 51.037v120.768H431zM123.448 88.722C97.774 88.722 81 105.601 81 127.724c0 21.658 16.264 39.002 41.455 39.002h.484c26.165.0 42.452-17.344 42.452-39.002-.485-22.092-16.241-38.954-41.943-39.002z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 12 on reddit" href="https://reddit.com/submit?url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f12%2f&title=Issue%2012"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zM446 265.638c0-22.964-18.616-41.58-41.58-41.58-11.211.0-21.361 4.457-28.841 11.666-28.424-20.508-67.586-33.757-111.204-35.278l18.941-89.121 61.884 13.157c.756 15.734 13.642 28.29 29.56 28.29 16.407.0 29.706-13.299 29.706-29.701.0-16.403-13.299-29.702-29.706-29.702-11.666.0-21.657 6.792-26.515 16.578l-69.105-14.69c-1.922-.418-3.939-.042-5.585 1.036-1.658 1.073-2.811 2.761-3.224 4.686l-21.152 99.438c-44.258 1.228-84.046 14.494-112.837 35.232-7.468-7.164-17.589-11.591-28.757-11.591-22.965.0-41.585 18.616-41.585 41.58.0 16.896 10.095 31.41 24.568 37.918-.639 4.135-.99 8.328-.99 12.576.0 63.977 74.469 115.836 166.33 115.836s166.334-51.859 166.334-115.836c0-4.218-.347-8.387-.977-12.493 14.564-6.47 24.735-21.034 24.735-38.001zM326.526 373.831c-20.27 20.241-59.115 21.816-70.534 21.816-11.428.0-50.277-1.575-70.522-21.82-3.007-3.008-3.007-7.882.0-10.889 3.003-2.999 7.882-3.003 10.885.0 12.777 12.781 40.11 17.317 59.637 17.317 19.522.0 46.86-4.536 59.657-17.321 3.016-2.999 7.886-2.995 10.885.008 3.008 3.011 3.003 7.882-.008 10.889zm-5.23-48.781c-16.373.0-29.701-13.324-29.701-29.698.0-16.381 13.328-29.714 29.701-29.714 16.378.0 29.706 13.333 29.706 29.714.0 16.374-13.328 29.698-29.706 29.698zM160.91 295.348c0-16.381 13.328-29.71 29.714-29.71 16.369.0 29.689 13.329 29.689 29.71.0 16.373-13.32 29.693-29.689 29.693-16.386.0-29.714-13.32-29.714-29.693z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 12 on facebook" href="https://facebook.com/sharer/sharer.php?u=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f12%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H342.978V319.085h66.6l12.672-82.621h-79.272v-53.617c0-22.603 11.073-44.636 46.58-44.636H425.6v-70.34s-32.71-5.582-63.982-5.582c-65.288.0-107.96 39.569-107.96 111.204v62.971h-72.573v82.621h72.573V512h-191.104c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 12 on whatsapp" href="https://api.whatsapp.com/send?text=Issue%2012%20-%20https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f12%2f"><svg viewBox="0 0 512 512"><path d="M449.446.0C483.971.0 512 28.03 512 62.554v386.892C512 483.97 483.97 512 449.446 512H62.554c-34.524.0-62.554-28.03-62.554-62.554V62.554c0-34.524 28.029-62.554 62.554-62.554h386.892zm-58.673 127.703c-33.842-33.881-78.847-52.548-126.798-52.568-98.799.0-179.21 80.405-179.249 179.234-.013 31.593 8.241 62.428 23.927 89.612l-25.429 92.884 95.021-24.925c26.181 14.28 55.659 21.807 85.658 21.816h.074c98.789.0 179.206-80.413 179.247-179.243.018-47.895-18.61-92.93-52.451-126.81zM263.976 403.485h-.06c-26.734-.01-52.954-7.193-75.828-20.767l-5.441-3.229-56.386 14.792 15.05-54.977-3.542-5.637c-14.913-23.72-22.791-51.136-22.779-79.287.033-82.142 66.867-148.971 149.046-148.971 39.793.014 77.199 15.531 105.329 43.692 28.128 28.16 43.609 65.592 43.594 105.4-.034 82.149-66.866 148.983-148.983 148.984zm81.721-111.581c-4.479-2.242-26.499-13.075-30.604-14.571-4.105-1.495-7.091-2.241-10.077 2.241-2.986 4.483-11.569 14.572-14.182 17.562-2.612 2.988-5.225 3.364-9.703 1.12-4.479-2.241-18.91-6.97-36.017-22.23C231.8 264.15 222.81 249.484 220.198 245s-.279-6.908 1.963-9.14c2.016-2.007 4.48-5.232 6.719-7.847 2.24-2.615 2.986-4.484 4.479-7.472 1.493-2.99.747-5.604-.374-7.846-1.119-2.241-10.077-24.288-13.809-33.256-3.635-8.733-7.327-7.55-10.077-7.688-2.609-.13-5.598-.158-8.583-.158-2.986.0-7.839 1.121-11.944 5.604-4.105 4.484-15.675 15.32-15.675 37.364.0 22.046 16.048 43.342 18.287 46.332 2.24 2.99 31.582 48.227 76.511 67.627 10.685 4.615 19.028 7.371 25.533 9.434 10.728 3.41 20.492 2.929 28.209 1.775 8.605-1.285 26.499-10.833 30.231-21.295 3.732-10.464 3.732-19.431 2.612-21.298-1.119-1.869-4.105-2.99-8.583-5.232z"/></svg></a><a target=_blank rel="noopener noreferrer" aria-label="share Issue 12 on telegram" href="https://telegram.me/share/url?text=Issue%2012&amp;url=https%3a%2f%2ffolded-weekly.netlify.app%2fposts%2f12%2f"><svg viewBox="2 2 28 28"><path d="M26.49 29.86H5.5a3.37 3.37.0 01-2.47-1 3.35 3.35.0 01-1-2.47V5.48A3.36 3.36.0 013 3 3.37 3.37.0 015.5 2h21A3.38 3.38.0 0129 3a3.36 3.36.0 011 2.46V26.37a3.35 3.35.0 01-1 2.47 3.38 3.38.0 01-2.51 1.02zm-5.38-6.71a.79.79.0 00.85-.66L24.73 9.24a.55.55.0 00-.18-.46.62.62.0 00-.41-.17q-.08.0-16.53 6.11a.59.59.0 00-.41.59.57.57.0 00.43.52l4 1.24 1.61 4.83a.62.62.0 00.63.43.56.56.0 00.4-.17L16.54 20l4.09 3A.9.9.0 0021.11 23.15zM13.8 20.71l-1.21-4q8.72-5.55 8.78-5.55c.15.0.23.0.23.16a.18.18.0 010 .06s-2.51 2.3-7.52 6.8z"/></svg></a></div></footer></article></main><footer class=footer><span>&copy; 2023 <a href=https://folded-weekly.netlify.app/>Folded Weekly</a></span>
<span>Powered by
<a href=https://gohugo.io/ rel="noopener noreferrer" target=_blank>Hugo</a> &
        <a href=https://git.io/hugopapermod rel=noopener target=_blank>PaperMod</a></span></footer><a href=#top aria-label="go to top" title="Go to Top (Alt + G)" class=top-link id=top-link accesskey=g><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentcolor"><path d="M12 6H0l6-6z"/></svg></a><script>let menu=document.getElementById("menu");menu&&(menu.scrollLeft=localStorage.getItem("menu-scroll-position"),menu.onscroll=function(){localStorage.setItem("menu-scroll-position",menu.scrollLeft)}),document.querySelectorAll('a[href^="#"]').forEach(e=>{e.addEventListener("click",function(e){e.preventDefault();var t=this.getAttribute("href").substr(1);window.matchMedia("(prefers-reduced-motion: reduce)").matches?document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView():document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView({behavior:"smooth"}),t==="top"?history.replaceState(null,null," "):history.pushState(null,null,`#${t}`)})})</script><script>var mybutton=document.getElementById("top-link");window.onscroll=function(){document.body.scrollTop>800||document.documentElement.scrollTop>800?(mybutton.style.visibility="visible",mybutton.style.opacity="1"):(mybutton.style.visibility="hidden",mybutton.style.opacity="0")}</script><script>document.getElementById("theme-toggle").addEventListener("click",()=>{document.body.className.includes("dark")?(document.body.classList.remove("dark"),localStorage.setItem("pref-theme","light")):(document.body.classList.add("dark"),localStorage.setItem("pref-theme","dark"))})</script></body></html>