---
title: "Issue 12"
date: 2022-02-09T08:50:49+01:00
showToc: true
TocOpen: true
draft: false
hidemeta: false
comments: false
ShowBreadCrumbs: false
ShowReadingTime: true
summary: |
  This week we cover protein evolution as seen by language models, molecular design by reinforcement learning, and molecular electronics to measure binding
cover:
    image: "img/12.png" # image path/url
    alt: "12" # alt text
    relative: false # when using page bundles set this to true
---

## [Evolutionary velocity with protein language models predicts evolutionary dynamics of diverse proteins](https://doi.org/10.1016/j.cels.2022.01.003)

Protein language models are revolutionizing protein bioinformatics. From the prediction of protein sequence features and the classification of protein families, to important contributions to protein structure prediction. In this paper, Hie *et al* go a step further and use them to model protein sequence evolution as a "walk through the protein sequence landscape". Based on the ESM-1b protein language model, which the authors say learnt general protein evolutionary patterns, they developed an approach that used distances in embedded space to build a sequence similarity network on top of which they map a "vector field" that translates into the evolutionary dynamics within the sequences analysed. The approach starts by the computation of the sequence embeddings for a target set of sequences using the language model. KNN as implemented in UMAP is then used to find the neighbours of each sequence in embedded space, mimicking a sequence similarity network. Then, the change in the language model pseudo-likelihood in each edge is used to assign a "velocity" vector whose direction represents an evolutionary transition. While it is not clear to me why this last step has the evolutionary meaning claimed, the authors demonstrate that for different protein families the internal structure of resulting network and the vector field generated captures their known evolutionary patterns. What makes this work even more exciting is that the method is made available as a python package ([evolocity](https://github.com/brianhie/evolocity)), which relies on [ScamPy](https://github.com/TommasoRonconi/scampy) and [anndata](https://github.com/theislab/anndata), two other packages that facilitate the analysis of large multidimensional annotated data. The package comes with multiple tutorials that can be run in Google Colab; this makes it easier to explore it for our favourite sets of sequences. But be warned: you may run into crashes due to memory overload in Colab. Nevertheless, it is exciting that such a tool is now made easily available and, while there is some amount of flexibility within the package, the expansion to other clustering methods and further language models would make it even more rich. 

## [Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning](https://arxiv.org/abs/2202.00658)
The authors describe an interesting molecule design method that combines Markov decision processes with machine learning. It consists  in a hierarchical agent that sequentially places molecular fragments with predefined shapes in 3D space. They compare this methodology with the intuition of an organic chemist who, when faced with a task to modify a molecule, would think of it in terms of adding new fragments and functional groups rather than single atoms. This method allows building molecules with more than 100 atoms, a task that other approaches that take the 3D structure into consideration usually fail to accomplish. Moreover, they suggest that it is possible to optimize different properties during the design process, like hydrophilicity or the affinity with a specific pocket. This makes this method extremely versatile, leading  to many potential applications such as: small molecule and peptide drug design or design of ligands and organocatalysts.

## [Molecular electronics sensors on a scalable semiconductor chip: A platform for single-molecule measurement of binding kinetics and enzyme activity](https://doi.org/10.1073/pnas.2112812119)

The future is now! The authors realise the 70s' dream of using single molecules as circuit elements, with a CMOS-integrated circuit chip consisting of an array of nanoelectrode-based single-molecule sensors "bridged" by a molecular wire between the nanoelectrodes. The biomolecule of interest is conjugated to the bridge molecule to act as a probe (a process that takes 10 seconds), the target molecule is added into the equation, and then the current through the bridge is monitored - if the target molecule interacts with the probe then this can be seen via the current pulses. And not just seen either -  Hidden Markov models are used to analyse the current signals, with bound and unbound being the hidden states, and by comparing the fraction of bound vs unbound across different target concentrations they obtain pretty accurate estimates for the binding affinity $K_d$ as well! The authors demonstrate an impressive variety of applications - DNA hybridization of complementary oligos (including determining the melting temperature of the DNA duplex, and showing sensitivity to introduced mismatches),  protein and small-molecule binding, DNA aptamer-target binding (shown using aptamers designed to bind SARS-CoV2 proteins), and enzyme activity (shown using a CRISPR/Cas system with the guide RNA targeting the Spike protein gene, and by distinguishing between DNA polymerase enzyme incorporation events) among others. They even show that adding a bunch of saliva or other highly complex contaminants doesn't mess up sensing, meaning it's perfect for diagnostic tests. Overall, the chip is sensitive and specific, can be multiplexed to target many kinds of interactions and molecules, highly scalable way past Moore's law, fast (seconds) and cheap (~1$). Looks like a bright future for obtaining high quality drug-target binding affinity data to train our methods!

## Figure of the Week

{{< figure src="img/12_fotw.png" >}}

**Source: [Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning](https://arxiv.org/abs/2202.00658)**

## Some more
- [Cryo-EM and artificial intelligence visualize endogenous protein community members](https://doi.org/10.1016/j.str.2022.01.001)
- [What stabilizes pre-folded structures in the intrinsically disordered α-helical binding motifs?](https://doi.org/10.1101/2022.01.28.478151)
- [Riemannian Geometry and Molecular Surfaces I: Spectrum of the Laplacian](https://arxiv.org/abs/2201.04230)
- [Computational Modeling of Molecular Structures Guided by Hydrogen-Exchange Data](https://doi.org/10.1021/jasms.1c00328)
- [Current structure predictors are not learning the physics of protein folding](https://doi.org/10.1093/bioinformatics/btab881)
- [AlphaDesign: A graph protein design method and benchmark on AlphaFoldDB](https://arxiv.org/abs/2202.01079)
- [DisEnrich: Database of Enriched Regions in Human Dark Proteome](https://doi.org/10.1093/bioinformatics/btac051)
- [EViS: An Enhanced Virtual Screening Approach Based on Pocket–Ligand Similarity](https://doi.org/10.1021/acs.jcim.1c00944)
- [Assessment of AlphaFold2 residue conformations for human proteins](https://www.biorxiv.org/content/10.1101/2022.01.28.478137v1)
- [ChemPlot, a Python library for chemical space visualization](https://doi.org/10.26434/chemrxiv-2021-3zv3k-v2)
- [SPOT-Contact-LM: Improving Single-Sequence-Based Prediction of Protein Contact Map using a Transformer Language Model](https://doi.org/10.1093/bioinformatics/btac053)
- A Springer book on [Artificial Intelligence in Drug Design](https://link.springer.com/book/10.1007/978-1-0716-1787-8)
- UniProt has a [Machine Learning Challenge](http://insideuniprot.blogspot.com/2022/02/the-uniprot-metal-binding-site-machine.html) aimed at creating computational methods to predict metal binding sites across the whole of UniProtKB
- Some resources on the latest hot topic, protein design via hallucination:  [A video tutorial](https://www.youtube.com/watch?v=2HmXwlKWMVs) and ColabFold support

{{< tweet 1489279054649991172 >}}
