---
title: "Issue 7"
date: 2021-12-20T19:07:18+01:00
showToc: true
TocOpen: true
draft: false
hidemeta: false
comments: false
ShowBreadCrumbs: false
ShowReadingTime: true
summary: |
  This week we look at ligand docking, SARS-CoV-2 variants, and enzyme active site flexibility.
cover:
    image: "img/7.png" # image path/url
    alt: "7" # alt text
    relative: false # when using page bundles set this to true
---


## [Leveraging nonstructural data to predict structures and affinities of proteinâ€“ligand complexes](https://doi.org/10.1073/pnas.2112621118)

The authors use the knowledge that even dissimilar ligands tend to bind to a given target protein in similar poses to improve computational binding pose and binding affinity prediction. Given a query ligand and target protein, [ComBind](https://github.com/drorlab/combind) finds other distinct ligands binding to that protein, docks all of them, and then picks poses for each ligand optimizing a potential shared across all the ligands, i.e. one that rewards protein-ligand interactions shared across different ligands. The same potentials are also used as a proxy for relative binding affinities, perfect for virtual screening. Quite a nice discussion comparing this to previous work and on integrating this concept with future improvements in docking and virtual screening.

## [Predictive profiling of SARS-CoV-2 variants by deep mutational learning](https://doi.org/10.1101/2021.12.07.471580)
The emergence of SARS-CoV 2 variants with multiple, combined mutations rises the concern that the virus may fall into an evolutionary path that promotes the escaping of multiple classes of neutralizing antibodies. In this paper, the authors combined yeast display with deep sequencing to interrogate a massive sequence space of combinatorial mutations, representing billions of variants of the receptor binding motif of the SARS-CoV2 Spike protein. This allowed them to generate a large, experimental labelled dataset of sequences that could then be used to train two machine learning models (a random forest and a long-short-term-memory recurrent neural network) that predict, at very high accuracy, whether a given variant would interact with the ACE2 enzyme and, if so, would escape four commercial neutralizing antibodies. The models may enable researchers to select candidate antibody therapeutics and could be used as a _de facto_ monitoring system, helping researchers to prospectively identify future variants with the highest likelihood of emergence.

## [Conformational variation in enzyme catalysis: A structural study on catalytic residues](https://doi.org/10.1101/2021.12.12.472283)
A very thorough analysis of the flexibility of catalytic residues from the Mechanism and Catalytic Site Atlas across over 900 enzyme families and across the entire PDB. Some conclusions: there's a high structural conservation of the active site across enzymes from the same family almost regardless of the sequence similarity; most active sites exhibit some degree of flexibility, half of which is accounted for by only side chain motion; the highest RMSD variability is observed when comparing ligand-free and ligand-bound active sites; "reactant" residues (proton donor, acceptor, and shuttle) are more rigid than "spectator" (electrostatic stabiliser, activator, steric role) residues; and, based on their RMSD distributions, active sites can be broadly divided into inherently rigid, inherently flexible, open/closed, and extensively variable. Look out for their [CSA-3D Python library](https://github.com/iriziotis/CSA-3D) which will help generalize analyses on catalytic sites like this one.

## Figure of the Week

{{< figure src="img/7_fotw.jpg" >}}

> Figure 1. Overview of deep mutational learning of the RBD for prediction of ACE2 binding and antibody escape.
> The RBD or the SARS-CoV-2 spike protein is expressed on the surface of yeast, mutagenesis libraries are designed on the receptor-binding motif (RBM-2) of the RBD, which are the sites of interaction with ACE2 and neutralizing antibodies (e.g., therapeutic antibody drugs). RBD libraries are screened by FACS for binding to ACE2 and neutralizing antibodies, both binding and non-binding (escape) populations are isolated and subjected to deep sequencing. Machine learning models are trained to predict binding status to ACE2 or antibodies based on RBD sequence. Machine learning models are then used to predict ACE2 binding and antibody escape on current and prospective variants and lineages.

**Source: [Predictive profiling of SARS-CoV-2 variants by deep mutational learning](https://doi.org/10.1101/2021.12.07.471580)**

## Some more
- [SARS-CoV-2 variants impact RBD conformational dynamics and ACE2 accessibility](https://doi.org/10.1101/2021.11.30.470470)
- [Synthon-based ligand discovery in virtual libraries of over 11 billion compounds](https://doi.org/10.1038/s41586-021-04220-9)
- [Learning physics confers pose-sensitivity in structure-based virtual screening](https://arxiv.org/abs/2110.15459)
- [InDeep: 3D fully convolutional neural networks to assist in silico drug design on protein-protein interactions](https://doi.org/10.1093/bioinformatics/btab849)
- [A novel sequence-based predictor for identifying and characterizing thermophilic proteins using estimated propensity scores of dipeptides](https://doi.org/10.1038/s41598-021-03293-w)
- [Constructing synthetic-protein assemblies from _de novo_ designed 310 helices](https://doi.org/10.1101/2021.12.11.471898)
- Version 3 of [Improved prediction of protein-protein interactions using AlphaFold2](https://doi.org/10.1101/2021.09.15.460468) 
	> ... "In short two major things have changed 1) We have introduced a pDockQ score that predicts the DockQ score for each protein-protein interactions. This improves both separation of good and bad models and interacting and non-interacting proteins. 2) We have added comparisons to (a) state of the art docking method and (b) alphafold-multimer."
- Version 2 of [Applications of AlphaFold beyond Protein Structure Prediction](https://doi.org/10.1101/2021.11.03.467194)
- Don't forget to check out the papers from the [Machine Learning in Structural Biology](https://www.mlsb.io/) workshop!